<!DOCTYPE html>
<!-- saved from url=(0041)https://liulab-dfci.github.io/publication -->
<html lang="en"><!-- Beautiful Jekyll | MIT license | Copyright Dean Attali 2016 --><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, viewport-fit=cover">

  <title>Publications</title>

  <meta name="author" content="Xiaole Shirley Liu Lab">

  
  <meta name="description" content="Publications in Liu Lab">
  

  <link rel="alternate" type="application/rss+xml" title="Liu Lab - Xiaole Shirley Liu Lab at DFCI" href="https://liulab-dfci.github.io/feed.xml">

  

  

  


  
    
      
  <link rel="stylesheet" href="./Publications_files/font-awesome.min.css">


    
  

  
    
      <link rel="stylesheet" href="./Publications_files/bootstrap.min.css">
    
      <link rel="stylesheet" href="./Publications_files/bootstrap-social.css">
    
      <link rel="stylesheet" href="./Publications_files/main.css">
    
  

  
    
      <link rel="stylesheet" href="./Publications_files/css">
    
      <link rel="stylesheet" href="./Publications_files/css(1)">
    
  

  

  

  

    <!-- Facebook OpenGraph tags -->
  

  
  <meta property="og:title" content="Publications">
  

   
  <meta property="og:description" content="Publications in Liu Lab">
  


  <meta property="og:type" content="website">

  
  <meta property="og:url" content="https://liulab-dfci.github.io/publication">
  <link rel="canonical" href="https://liulab-dfci.github.io/publication">
  

  


  <!-- Twitter summary cards -->
  <meta name="twitter:card" content="summary">
  <meta name="twitter:site" content="@">
  <meta name="twitter:creator" content="@">

  
  <meta name="twitter:title" content="Publications">
  

  
  <meta name="twitter:description" content="Publications in Liu Lab">
  

  

  

  
  <link rel="icon" href="https://liulab-dfci.github.io/img/favicon.ico" type="image/x-icon">
  <link rel="shortcut icon" href="https://liulab-dfci.github.io/img/favicon.ico" type="image/x-icon">
</head>


  <body>

    

  
    <nav class="navbar navbar-default navbar-fixed-top navbar-custom top-nav-short">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#main-navbar">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button><a class="navbar-brand" href="https://liulab-dfci.github.io/">Liu Lab</a></div>

    <div class="collapse navbar-collapse" id="main-navbar">
      <ul class="nav navbar-nav navbar-right">
          <li><a href="https://liulab-dfci.github.io/">Home</a></li>
          <li><a href="https://liulab-dfci.github.io/software">Software</a></li>
          <li><a href="https://liulab-dfci.github.io/news">News &amp; Events</a></li>
          <li><a href="https://liulab-dfci.github.io/people">People</a></li>
          <li><a href="https://liulab-dfci.github.io/publication">Publications</a></li>
          <li><a href="https://liulab-dfci.github.io/teaching">Teaching</a></li>
          <li><a href="https://liulab-dfci.github.io/careers">Careers</a></li>
          <li><a href="https://liulab-dfci.github.io/contact">Contact</a></li></ul>
    </div>

	

  </div>
</nav>


        <div class="intro-header">
  <div role="main" class="container">
    <h1 id="publications">Publications</h1>

<p>You can visit <a href="https://scholar.google.com/citations?user=8XNfVucAAAAJ">Google Scholar</a> for more details<br>
Click here for Complete <a href="https://www.ncbi.nlm.nih.gov/myncbi/xiaole.liu.1/bibliography/public/">PubMed Bibliography</a><br>
# Co-first authors; * Co-corresponding authors</p>

<h2 id="2023">2023</h2>
<ul>
  <li>Yang L, Wang J, Altreuter J, Jhaveri A, Wong CJ, Song L, Fu J, Taing L, Bodapati S, Sahu A, Tokheim C, Zhang Y, Zeng Z, Bai G, Tang M, Qiu X, Long HW, Michor F, Liu Y, <strong>Liu XS</strong>. 2023. Tutorial: integrative computational analysis of bulk RNA-sequencing data to characterize tumor immunity using RIMA. Nat Protoc. 18(8):2404-2414. <br>
<a href="https://pubmed.ncbi.nlm.nih.gov/37391666/">[Abstract]</a></li>
  <li>Zhang Y#, Xiang G, Jiang AY, Lynch A, Zeng Z, Wang C, Zhang W, Fan J, Kang J, Gu SS, Wan C, Zhang B, <strong>Liu XS</strong>*, Brown M*, Meyer CA*.2023. MetaTiME integrates single-cell gene expression to characterize the meta-components of the tumor immune microenvironment. Nat Commun. 2023 May 6;14(1):2634. <br>
<a href="https://pubmed.ncbi.nlm.nih.gov/37149682/">[Abstract]</a></li>
  <li>Soto-Feliciano YM#, Sánchez-Rivera FJ#, Perner F#, Barrows DW, Kastenhuber ER, Ho YJ, Carroll T, Xiong Y, Anand D, Soshnev AA, Gates L, Beytagh MC, Cheon D, Gu S, <strong>Liu XS</strong>, Krivtsov AV, Meneses M, de Stanchina E, Stone RM, Armstrong SA*, Lowe SW*, Allis CD*. 2023. A Molecular Switch between Mammalian MLL Complexes Dictates Response to Menin-MLL Inhibition. Cancer Discov. 13(1):146-169. <br>
<a href="https://pubmed.ncbi.nlm.nih.gov/36264143/">[Abstract]</a></li>
  <li>Theodoris CV, Xiao L, Chopra A, Chaffin MD, Al Sayed ZR, Hill MC, Mantineo H, Brydon EM, Zeng Z, <strong>Liu XS</strong>, Ellinor PT. 2023. Transfer learning enables predictions in network biology. Nature. 618(7965):616-624. <br>
<a href="https://pubmed.ncbi.nlm.nih.gov/37258680/">[Abstract]</a></li>
  <li>Ito Y, Pan D, Zhang W, Zhang X, Juan TY, Pyrdol JW, Kyrysyuk O, Doench JG, <strong>Liu XS</strong>, Wucherpfennig KW. 2023. Addressing Tumor Heterogeneity by Sensitizing Resistant Cancer Cells to T cell-Secreted Cytokines. Cancer Discov. 13(5):1186-1209. <br>
<a href="https://pubmed.ncbi.nlm.nih.gov/36811466/">[Abstract]</a></li>
  <li>Song L, Bai G, <strong>Liu XS</strong>, Li B, Li H. 2023. Efficient and accurate KIR and HLA genotyping with massively parallel sequencing data. Genome Res. 33(6):923-931. <br>
<a href="https://pubmed.ncbi.nlm.nih.gov/37169596/">[Abstract]</a></li>
  <li>Sahu A, Wang X, Munson P, Klomp JPG, Wang X, Gu SS, Han Y, Qian G, Nicol P, Zeng Z, Wang C, Tokheim C, Zhang W, Fu J, Wang J, Nair NU, Rens JAP, Bourajjaj M, Jansen B, Leenders I, Lemmers J, Musters M, van Zanten S, van Zelst L, Worthington J, Liu JS, Juric D, Meyer CA, Oubrie A, <strong>Liu XS</strong>, Fisher DE, Flaherty KT. 2023. Discovery of Targets for Immune-Metabolic Antitumor Drugs Identifies Estrogen-Related Receptor Alpha. Cancer Discov. 13(3):672-701. <br>
<a href="https://pubmed.ncbi.nlm.nih.gov/36745048/">[Abstract]</a></li>
</ul>

<h2 id="2022">2022</h2>
<ul>
  <li>Zeng Z#, Gu SS#, Ouardaoui N, Tymm C, Yang L, Wong CJ, Li D, Zhang W, Wang X, Weirather JL, Rodig SJ, Hodi FS*, Brown M*, <strong>Liu XS</strong>*. 2022. Hippo signaling pathway regulates cancer cell-intrinsic MHC-II expression. Cancer Immunol Res. CIR-22-0227. <br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/36219700">[Abstract]</a></li>
  <li>Zeng Z#, Gu SS#, Wong CJ, Yang L, Ouardaoui N, Li D, Zhang W, Brown M, <strong>Liu XS</strong>*. 2022. Machine learning on syngeneic mouse tumor profiles to model clinical immunotherapy response. Sci Adv. 8(41):eabm8564 <br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/36240281">[Abstract]</a></li>
  <li>Lynch AW#, Theodoris CV#, Long HW, Brown M, <strong>Liu XS</strong>*, Meyer CA*. 2022. MIRA: joint regulatory modeling of multimodal expression and chromatin accessibility in single cells. Nat Methods. 19(9):1097-1108. <br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/36068320">[Abstract]</a></li>
  <li>Song L, Ouyang Z, Cohen D, Cao Y, Altreuter J, Bai G, Hu X, Livak KJ, Li H, Tang M, Li B, <strong>Liu XS</strong>*. 2022. Comprehensive Characterizations of Immune Receptor Repertoire in Tumors and Cancer Immunotherapy Studies. Cancer Immunol Res. 10(7):788-799. <br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/35605261">[Abstract]</a></li>
  <li>Hu X*, <strong>Liu XS</strong>*. 2022. A high-resolution view of intra-tumoral B cell immunity. Immunity. 55(3):387-389. <br>
<a href="https://pubmed.ncbi.nlm.nih.gov/35263565/">[Abstract]</a></li>
  <li>Liao Y#, Chen CH#, Xiao T, Peña Avalos B, Dray EV, Cai C, Gao S, Shah N, Zhang Z, Feit A, Xue P, Liu Z, Yang M, Ji HL, Xu H, Li W, Mei X, Pierre RS, Shu S, Fei T, Duarte M, Xhao J, Badner JE, Polyak K, Kantoff PW, Long H, Balk SP, <strong>Liu XS</strong>*, Brown M*, Xu K*. 2022. Inhibition of EZH2 transactivation function sensitizes solid tumors to genotoxic stress. Proc Natl Acad Sci USA. 119(3):e2105898119 <br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/35031563">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PNAS119_2022.pdf">[Full Paper]</a></li>
  <li>Choi YS, Erlich TH, von Franque M, Rachmin I, Flesher JL, Schiferle EB, Zhang Y, Pereira da Silva M, Jiang A, Dobry AS, Su M, Germana S, Lacher S, Freund O, Feder E, Cortez JL, Ryu S, Babila Propp T, Samuels YL, Zakka LR, Azin M, Burd CE, Sharpless NE, <strong>Liu XS</strong>, Meyer C, Austen WG Jr, Bojovic B, Cetrulo CL Jr, Mihm MC, Hoon DS, Demehri S, Hawryluk EB, Fisher DE. 2022. Topical therapy for regression and melanoma prevention of congenital giant nevi. Cell. 185(12):2071-2085.e12. <br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/35561684">[Abstract]</a></li>
  <li>Aubrey BJ, Cutler JA, Bourgeois W, Donovan KA, Gu S, Hatton C, Perlee S, Perner F, Rahnamoun H, Theall ACP, Henrich JA, Zhu Q, Nowak RP, Kim YJ, Parvin S, Cremer A, Olsen SN, Eleuteri NA, Pikman Y, McGeehan GM, Stegmaier K, Letai A, Fischer ES, <strong>Liu XS</strong>, Armstrong SA. 2022. IKAROS and MENIN coordinate therapeutically actionable leukemogenic gene expression in MLL-r acute myeloid leukemia. Nat Cancer. 3(5):595-613. <br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/35534777">[Abstract]</a></li>
  <li>Obradovic A, Graves D, Korrer M, Wang Y, Roy S, Naveed A, Xu Y, Luginbuhl A, Curry J, Gibson M, Idrees K, Hurley P, Jiang P, <strong>Liu XS</strong>, Uppaluri R, Drake CG, Califano A, Kim YJ. 2022. Immunostimulatory Cancer-Associated Fibroblast Subpopulations Can Predict Immunotherapy Response in Head and Neck Cancer. Clin Cancer Res. 28(10):2094-2109. <br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/35262677">[Abstract]</a></li>
  <li>Zhou L#, Zeng Z#, Egloff AM, Guo F, Campbell KM, Du P, Fu J, Zolkind P, Ma X, Zhang Z, Zhang Y, Wang X, Gu S, Riley R, Nakahori Y, Keegan J, Haddad R, Schoenfeld JD, Griffith O, Manguso RT, Lederer JA, <strong>Liu XS</strong>, Uppaluri R*. 2022. Checkpoint blockade-induced CD+ Tcell differentiation in head and neck cancer responders. J Immunother Cancer. 10(1):e004034. <br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/35058328">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Zhou_JITC_2022.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2021">2021</h2>
<ul>
  <li>Zhang Y#, Liu Y, <strong>Liu XS</strong>*. 2021. Neural network architecture search with AMBER. Nat Machine Intell 3:372-373.<br>
<a href="https://liulab-dfci.github.io/resources/publications/CompGen3_372.pdf">[Full Paper]</a></li>
  <li>Liu J#, Tokheim C#, Lee JD#, Gan W, North BJ, <strong>Liu XS</strong>*, Pandolfi PP*, Wei W*. 2021. Genetic fusions favor tumorigenesis through degron loss in oncogenes. Nat Commun 12(1):6704.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/34795215">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatCommun12_6704.pdf">[Full Paper]</a></li>
  <li>Zhang H#, Song L#, Wang X, Cheng H, Wang C, Meyer CA, Liu T, Tang M, Aluru S, Yur F, <strong>Liu XS</strong>*, Li H*. 2021. Fast alignment and preprocessing of chromatin profiles with Chromap. Nat Commun 12(1):1-6.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/34772935">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatCommun12_1.pdf">[Full Paper]</a></li>
  <li>Wang X#, Tokheim C, Gu SS, Wang B, Tang Q, Li Y, Traugh N, Zeng Z, Zhang Y, Li Z, Shang B, Fu J, Xiao T, Li W, Meyer CA, Chu J, Jiang P, Cejas P, Lim K, Long H, Brown M*, <strong>Liu XS</strong>*. 2021. In vivo CRISPR screens identify the E3 ligase Cop1 as a modulator of macrophage infiltration and cancer immunotherapy target. Cell 184(21):5357-5374.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/34582788">[Abstract]</a> 
<a href="https://liulab-dfci.github.io/resources/publications/Cell184_5357.pdf">[Full Paper]</a></li>
  <li>Zeng Z#, Wong C#, Yang L#, Ouardaoui N, Li D, Zhang W, Gu SS, Zhang Y, Liu Y, Wang Z, Fu J, Zhou L, Zhang B, Kim S, Yates KB, Brown M, Freeman GJ, Uppaluri R, Manguso, <strong>Liu XS</strong>. 2021. TISMO: syngeneic mouse tumor database to model tumor immunity and immunotherapy response. Nuc Acids Res. <br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/34534350">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NucAcidRes_1_2021.pdf">[Full Paper]</a></li>
  <li>Song L#, Cohen D, Ouyang Z, Cao Y, Hu X, <strong>Liu XS</strong>. 2021. TRUST4: immune repertoire reconstruction from bulk and single-cell RNA-seq data. Nat Methods 18:627-630.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/33986545">[Abstract]</a> 
<a href="https://liulab-dfci.github.io/resources/publications/NatMed18_627.pdf">[Full Paper]</a>
<a href="https://github.com/liulab-dfci/TRUST4">[Code]</a></li>
  <li>Gu S#, Zhang W, Wang X, Jiang P, Traugh N, Li Z, Meyer C, Stewig B, Xie Y, Bu X, Manos MP, Font-Tello A, Gjini E, Lako A, Lim K, Conway J, Tewari AK, Zeng Z, Das Sahu A, Tokheim C, Weirather JL, Fu J, Zhang Y, Kroger B, Liang JH, Cejas P, Freeman GJ, Rodig S, Long HW, Gewurz BE, Hodi FS, Brown M*, <strong>Liu XS</strong>*. 2021. Therapeutically increasing MHC-I expression potentiates immune checkpoint blockade. Cancer Discov 11:1-18.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/33589424">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CancDisc_11_2021.pdf">[Full Paper]</a>
<a href="https://github.com/WubingZhang/MHCI_TRAF3">[Data and Code]</a></li>
  <li>Tokheim C#, Wang X, Timms RT, Zhang B, Mena EL, Wang B, Chen C, Ge J, Zhang W, Elledge SJ*, Brown M*, <strong>Liu XS</strong>*. 2021. Systematic characterization of mutations altering protein degradation in human cancers. Mol Cell 81:1-17.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/33567269">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/MolCell81_1.pdf">[Full Paper]</a>
<a href="https://deepdegron.readthedocs.io/en/latest/index.html">[Code and Documentation]</a></li>
  <li>Qiu X, Feit AS, Feiglin A, Xie Y, Taing L, Perkins J, Gu S, Li Y, Cejas P, Zhou N, Jeselsohn R, Brown M, <strong>Liu XS</strong>, Long HW. 2021. COBRA: Containerized bioinformatics workflow for reproducible ChIP/ATAC-seq analysis. Genomics Proteomics Bioinformatics. <br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/34284136">[Abstract]</a>
<a href="https://www.sciencedirect.com/science/article/pii/S1672022921001546">[Full Paper]</a></li>
  <li>Taing L#, Bai G#, Cousins C#, Cejas P, Oiu X, Herbert ZT, Brown M, Meyer CA, <strong>Liu XS</strong>, Long HW*, Tang M*. 2021. CHIPS: A snakemake pipeline for quality control and reproducible processing of chromatin profiling data (version 1; peer review: awaiting peer review). F1000 Rsch 10:517.<br>
<a href="https://f1000research.com/articles/10-517/v1">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/F1000Rsch10_517.pdf">[Full Paper]</a>
<a href="https://github.com/liulab-dfci/CHIPS">[Code]</a></li>
  <li>Wu SZ, Al-Eryani G, Roden DL, Junakar S, Harvey K, Andersson A et al. 2021.  A single-cell and spatially resolved atlas of human breast cancers. Nat Genet 53(9):1334-1337.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/34493872/">[Abstract]</a>
<a href="https://drive.google.com/file/d/1HPrENRrDnVd0pZkfZKoEDyBB5zr3fxl-/view?usp=sharing">[Full Paper]</a></li>
  <li>Penter L, Zhang Y, Saveli A Huang T, Cieri N, thrash EM, Kim-Schulze S, Jhaveri A, Fu J, Ranasinghe S, Li S, Zhang W, Hathaway ES, Nazzaro M, Kim HT, Chen H, Thurin M, Rodig SJ, Severgnini M, Cibulskis C, Gabriel S, Livak KJ, Cutler C, Antin JH, Nikiforow S, Koreth J, Ho VT, Armand P, Ritz J, Streicher H, Neuberg D, Hodi FS, Gnjatic S, Soiffer RJ, <strong>Liu XS</strong>, Davids MS*, Bachireddy P*, Wu CJ*. 2021. Molecular and cellular features of CTLA-4 blockade for relapsed myeloid malignancies after transplantation. Blood 137(23):3212-17.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/33720354">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Blood137_2021.pdf">[Full Paper]</a></li>
  <li>Kumar S#, Zeng Z#, Bagati A, Tay RE, Sanz LA, Hartono SR, Ito Y, Abderazzaq F, Hatchi E, Jiang P, Cartwright ANR, Olawoyin O, Mathewson ND, Pyrdol JW, Li MZ, Doench JG, Booker MA, Tolstorukov MY, Elledge SJ, Chedin F, <strong>Liu XS</strong>*, Wucherpfennig KW*. 2021. CARM1 inhibition enables immunotherapy of resistant tumors by dual action on tumor cells and T cells. Cancer Discov 11:2051-71.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/33707234">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CancerDiscov_2051_Kumar.pdf">[Full Paper]</a></li>
  <li>Chen HX, Song M, Maecker HT, Gnjatic S, Patton D, Lee JJ, Adam SJ, Moravec R, <strong>Liu XS</strong>, Cerami E, Lindsay J, Hodi FS, Wu C, Wistuba II, Al-Atrash G, Bernatchez C, Bendall SC, Hewitt SM, Sharon E, Streicher H, Enos RA, Bowman MD, Tatard-Leitman VM, Sanchez-Espiridion B, Ranasinghe S, Pichavent M, Del Valle DM, Yu J, Janssens S, Peterson-Klaus J, Rowe C, Bongers G, Jenq RR, Chang CC, Abrams JS, Mooney M, Doroshow JH, Harris LN, Thurin M. 2021. Network for biomarker immunoprofiling for cancer immunotherapy: cancer immune monitoring and analysis centers and cancer immunologic Data Commons (CIMAC-CIDC). Clin Cancer Res 27(18):5038-5048.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/33419780">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/ClinCancerRes_5038.pdf">[Full Paper]</a></li>
  <li>Mathewson ND, Ashenberg O, Tirosh I, Gritsch S, Perez EM, Marx S, Jerby-Aron L, Chanoch-Myers R, Hara T, Richman AR, Ito Y, Pyrdol J, Friedrich M, Schumann K, Poitras MJ, Gokhale PC, Gonzales Castro LN, Shore ME, Hebert CM, Shaw B, Cahil HL, Drummond M, Zhang W, Olawoyin O, Wakimoto H, Rozenblatt-Rosen O, Brastianos PK, <strong>Liu XS</strong>, Jones PS, Cahill DP, Frosch MP, Louis DN, Freeman GJ, Ligon KL, Marson A, Chiocca EA, Reardon DA*, Regev A*, Suvà ML*, Wucherpfennig KW*. 2021. Inhibitory CD-161 receptor identified in glioma-infiltrating T cells by single-cell analysis. Cell 184:1-18.<br> 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/33592174">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Cell184_1.pdf">[Full Paper]</a></li>
  <li>Cejas P, Xie Y, Font-Tello A, Lim K, Syamala S, Qiu X, Tewari AK, Shah N, Nguyen HM, Patel RA, Brown L, Coleman I, Hackeng WM, Brosens L, Dreijerink KMA, Ellis L, Abou Alaiwi S, Seo J, Baca S, Beltran H, Khani F, Pomerantz M, Dall'Agnese A, Crowdis J, Van Allen EM, Bellmunt J, Morrisey C, Nelson PS, DeCaprio J, Farago A, Dyson N, Drapkin B, <strong>Liu XS</strong>, Freedman M, Haffner M, Corey E, Brown M*, Long HW*. 2021. Subtype heterogeneity and epigenetic convergence in neuroendocrine prostate cancer. Nat Commun 12:5775.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/34599169">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatCommun12_5775.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2020">2020</h2>
<ul>
  <li>Meyer CA, <strong>Liu XS</strong>. 2020. Computaional approaches to modeling transcription factor activity and gene regulation. Trends Biochem Sci 45(12):1094-5.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/33051080">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/TrendsBiochemSci45_1094.pdf">[Full Paper]</a></li>
  <li>Gu S#, Wang X#, Hu X#, Jiang P, Li Z, Traugh N, Bu X, Tang Q, Wang C, Zeng Z, Fu J, Meyer CA, Zhang Y, Cejas P, Lim K, Wang J, Zhang W, Tokheim C, Sahu A,, Xing X, Kroger B, Ouyang Z, Long H, Freeman G*, Brown M*, <strong>Liu XS</strong>*. 2020. Clonal tracing reveals diverse patters of response to immune checkpoint blockade. Genome Biol 21(1):263.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/33059736">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenomeBiol21_263.pdf">[Full Paper]</a></li>
  <li>Li B, Li T, Liu JS, <strong>Liu XS</strong>. 2020. Computational deconvolution of tumor-infiltrating immune components with bulk tumor gene expression data. Methods Mol Biol 2120:249-262.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32124325">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/MethMolBiol2120_249.pdf">[Full Paper]</a></li>
  <li>Cader FZ#, Hu F#, Goh W, Wienand K, Ouyang J, Mandato E, Redd R, Lawton L, Chen PH, Weirather J, Schackman RC, Li B, Ma W, Armand P, Rodig SJ, Neuberg D, <strong>Liu XS</strong>*, Shipp M*. 2020. A peripheral immune signature of responsiveness to PD-1 blockade in patients with classical Hodgkin lymphoma. NatMed 26(9):1468-1479.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32778827">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatMed_Cader_2020.pdf">[Full Paper]</a></li>
  <li>Wang C#, Sun D#, Huang X, Wan C, Li Z, Han Y, Qin Q, Fan J, Qiu X, Xie Y, Meyer C, Brown M, Tang M, Long H, Liu T*, <strong>Liu XS</strong>*. 2020. Integrative analyses of single-cell transcriptome and regulome using MAESTRO. Genome Biol 21:198.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32767996">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenBiol21_198.pdf">[Full Paper]</a>
<a href="https://github.com/liulab-dfci/MAESTRO">[Code]</a></li>
  <li>Zhang J#, Lee D#, Dhiman V#, Jiang P#, Xu J#, McGillivray P#, Yang H#, Liu J, Meyerson W, Clarke D, Gu M, Li S, Lou S, Xu J, Lochovsky L, Ung M, Ma L, Yu S, Cao Q, Harmanci A, Yan KK, Sethi A, Gürsoy G, Schoenberg MR, Rozowsky J, Warrell J, Emani P, Yang YT, Galeev T, Kong X, Liu S, Li X, Krishnan J, Feng Y, Rivera-Mulia JC, Adrian J, Broach JR, Bolt M, Moran J, Fitzgerald D, Dileep V, Liu T, Mei S, Sasaki T, Trevilla-Garcia C, Wang S, Wang Y, Zang C, Wang D, Klein RJ, Snyder M, Gilbert DM, Yip K, Cheng C, Yue F*, <strong>Liu XS</strong>*, White KP*, Gerstein M*. 2020. An integrative ENCODE resource for cancer genomics. Nat Commun 11:3696.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32728046">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatCommun11_3696.pdf">[Full Paper]</a></li>
  <li>Li Z#, Wang B#, Gu S, Peng J, Sahu A, Chen C, Han T, Shi S, Wang X, Traugh N, Liu H, Liu Y, Wu Q, Brown M, Xiao T*, Boland G*, <strong>Liu XS</strong>*. 2020. CRISPR screens identify essential cell growth mediators in BRAF inhibitor-resistant melanoma. Genomics Proteomics Bioinformatics 18(1):26-40.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32413516">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Li_GenProBio_2020.pdf">[Full Paper]</a></li>
  <li>Chen C#, Zheng R, Tokheim C, Dong X, Fan J, Wan C, Tang Q, Brown M, Liu JS, Meyer C*, <strong>Liu XS</strong>*. 2020. Determinants of transcription factor regulatory range. Nat Commun 11:2472.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32424124">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Chen_NatComm_2020.pdf">[Full Paper]</a></li>
  <li>Shi S#, Gu S#, Han T, Zhang W, Huang L, LiZ, Pan D, Fu J, Ge J, Brown M, Zhang P, Jiang P*, Wucherpfennig KW*, <strong>Liu XS</strong>*. 2020. Inhibition of MAN2A1 enhances tumor response to anti-PD-L1. Clin Cancer Res 28:OF1-OF13<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32723834">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CCR_MAN2A1_2020.pdf">[Full Paper]</a></li>
  <li>Fu J, Li K, Zhang W, Wan C, Zhang J*, Jiang P*, <strong>Liu XS</strong>*. 2020. Large-scale public data reuse to model immunotherapy response and resistance. Genome Med 12(1):21.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32102694">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Fu_GenMed_2020.pdf">[Full Paper]</a></li>
  <li>Qin Q#, Fan J#, Zheng R, Wan C, Mei S, Wu Q, Sun H, Brown M, Zhang J*, Meyer CA*, <strong>Liu XS</strong>*. 2020. Lisa: inferring transcriptional regulators through integrative modeling of public chromatin accessibiltiy and ChIP-seq data. Genome Biol 21(1):32.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32033573">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Qin_GenBiol_2020.pdf">[Full Paper]</a>
<a href="http://lisa.cistrome.org/">[Website]</a>
<a href="https://github.com/liulab-dfci/lisa2">[Code for LISA2]</a></li>
  <li>Li, T#, Fu J#, Zeng Z, Cohen D, Li J, Chen Q, Li B, <strong>Liu, XS</strong>*. 2020. TIMER2.0 for analysis of tumor-infiltrating immune cells. Nucleic Acids Res 48(W1):W509-W514. <br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32442275">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Li_NucAcidsRsh_2020.pdf">[Full Paper]</a></li>
  <li>Zeng Z#, Fu J#, Cibulskis C, Jhivari A, Gumbs C, Das B, Sanchez-Espiridion B, Janssens S, Taing L#, Wang J, Lindsay J, Vilimas T, Zhang J, Tokheim C, Sahu AD, Jiang P, Yan C, Y. Duose D, Cerami E, Chen L, Cohen D, Chen QR, Enos RA, Huang X, Lee JJ, Liu Y, Neuberg DS, Nguyen C, Patterson C, Sarkar S, Shukla SA, Tang M, Tsuji J, Uduman M, Wang X, Jason L Weirather, Jijun Yu, Joyce Yu, Zhang J, Zhang J, Meerzaman D, Thurin M, Futreal PA, Karlovich C, Gabriel SB, Wistuba II<em>, <strong>Liu XS</strong>*, Wu C</em>. 2020. Cross-site concordance evaluation of tumor DNA and RNA sequencing platforms for the CIMAC-CIDC network. Clin Cancer Res. 27(18):5049-5061. <br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/33323402">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CCR_5049.pdf">[Full Paper]</a></li>
  <li>Bagati A, Kumar A, Jiang P, Pyrdol J, Zou AE, Godicelj A, Mathewson ND, Cartwright ANR, Cejas P, Brown M, Giobbie-Hurder A, Dillon D, Agudo J, Mittendorf E, <strong>Liu XS</strong>, Wucherpfennig KW. 2020. Integrin αvβ6-TGFβ-SOX4 pathway drives immune evasion in triple-negative breast cancer. Cancer Cell 39:1-14.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/33385331">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CaCell_39.pdf">[Full Paper]</a></li>
  <li>Abou Alaiwi S, Nassar AH, Zie W, Bakouny Z, Berchuck JE, Braun DA, Baca SC, Nuzzo PV, Flippot R, Mouhieddine TH, Spurr LF, Li YY, Li T, Flaifel A, Steinharter JA, Margolis CA, Vokes NI, Du H, Shukla SA, Cherniack AD, Sonpavde G, Haddad RI, Awad MM, Giannakis M, Hodi FS, <strong>Liu XS</strong>, Signoretti S, Kadoch C, Freedman ML, KWiatkowski DJ, Van Allen EM, Choueiri TK. 2020. Mammalian SWI/SNF complex genomic alterations and immune checkpoint blockade in solid tumors. 2020 Cancer Immunol Res 8(8):1075-1084.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32321774">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Alaiwi_CAImmunoRsch_2020.pdf">[Full Paper]</a></li>
  <li>The ENCODE Project Consortium (List of authors provided in Supp. Info. section). 2020. Expanded encyclopaedias of DNA elements in the human and mouse genomes. Nature 583:699-710.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32728249">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Nature20_Expan583.pdf">[Full Paper]</a></li>
  <li>The ENCODE Project Consortium (List of authors provided in Supp. Info. section). 2020. Perspectives on ENCODE. Nature 583:693-698.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32728248">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Nature20_Persp583.pdf">[Full Paper]</a></li>
  <li>Li X et al. (Complete list of authors available online). 2020. Dynamic incorporation of multiple in silico functional annotations empowers rare variant association analysis of large whole-genome sequencing studies at scale. Nat Genet 52(9):969-983. <br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32839606">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatGenet52_969.pdf">[Full Paper]</a></li>
  <li>Gao Y, Nihira NT, Bu X, Chu C, Zhang J, Kolodziejczyk A, Fan Y, Chan NT, Ma L, Liu J, Wang D, Dai X, Liu H, Ono M, Nakanishi A, Inuzuka H, North BJ, Huang YH, Sharma S, Geng Y, Xu W, <strong>Liu XS</strong>, Li L, Miki Y, Sicinski P, Freeman GJ, Wei W. Acetylation-dependent regulation of PD-L1 nuclear translocation dictates the efficacy of anti-PD-1 immunotherapy.  Nat Cell Biol 22(9):1064-1075.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32839551">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatCellBiol22_1064.pdf">[Full Paper]</a></li>
  <li>Wu SZ, Roden DL, Wang C, Holliday H, Harvey K, Cazet AS, Murphy KJ, Pereira B, Al-Eryani G, Bartoniecek N, Hou R, Torpy JR, Junakar S, Chan C, Lan CE, Hui MN, Gluch L, Beith J, Parker A, Robbins E, Segara D, Mak C, Cooper C, Warrier S, Forrest A, Powell J, O'Toole S, Cox T, Timpson P, Lim E, <strong>Liu XS</strong>, Swarbrick A. 2020. Stromal cell diversity associated with immune evasion in human triple-negative breast cancer. EMBO Jrnl 13:1-20.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32790115">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/EMBO20_BRCA_scRNAseq.pdf">[Full Paper]</a></li>
  <li>Liu X, Liu T, Shang Y, Dai P, Zhang W, Lee BJ, Huang M, Yang D, Wu Q, Liu DL, Zheng X, Zhou BO, Dong J, Yeap L, Hu J, Xiao T, Zha S, Casellas R, <strong>Liu XS</strong>, Meng F. 2020. ERCC6L2 promotes DNA orientation-specific recombination in mammalian cells. Cell Res 30(9):732-744.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32355287">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Liu_CellRes_2020.pdf">[Full Paper]</a></li>
  <li>Shu S, Wu HJ, Ge JY, Seid R, Harris IS, Jovanovic B, Murphy K, Wang B, Qui X, Endress JE, Reyes J, Lim K, Font-Tello A, Syamala S, Ziao T, Reddy Chilamakuri CS, Papachristou EK, D'Santos C, Anand J, Hinohara K, Li W, McDonald TO, Luoma A, Modiste RJ, Nguyen QD, Michel B, Cejas P, Kadoch C, Jaffe JD, Wucherpfennig KW, Qi J, <strong>Liu XS</strong>, Long H, Brown M, Carroll JS, Brugge JS, Bradner J, Michor F, Polyak K. 2020. Synthetic lethal and resistance interaction with BET Bromodomain inhibitors in triple-negative breast cancer. Mol Cell 78(6):1096-1113.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32416067">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Shu_MolCell_2020.pdf">[Full Paper]</a></li>
  <li>Wan C, Keany M, Dong H, Al-Alem L, Pandya U, Saso S, Boehnke K, Lynch K, Xu R, Zarrella D, Gu S, Cejas P, Lim K, Long H, Elias K, Horowitz N, Reitmate C, Muto M, Worley M, Berkowitz R, Matulonis U, Nucci M, Crum C, Rueda B, Brown M, <strong>Liu XS</strong>, Hill S. 2020. Enhanced efficacy of simultaneous PD-1 and PD-L1 immune checkpoint blockade in high grade serous ovarian cancer. Cancer Res 81(1):158-73.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/33158814">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CancerRes_81.pdf">[Full Paper]</a></li>
  <li>Chu SH, Chabon JR, Matovina CN, Chen BR, Zhang J, Kuman V, Xiong Y, Callen E, Hung PJ, Feng Z, Koche RP, <strong>Liu XS</strong>, Chaudhuri J, Nussenzweig Z, Sleckman BP, Armstrong SA*. 2020. Loss of H3K36 Methyltransferase SETD2 impairs V(E)J recombination during lymphoid development. iScience 23(3):100941. <br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/32169821">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Chu_iScience_2020.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2019">2019</h2>
<ul>
  <li>Fei T#, Li W#, Peng J#, Xiao T, Chen CH, Wu A, Huang J, Zang C, <strong>Liu XS</strong>*, Brown M*. 2019. Deciphering essential cistromes using genome-wide CRISPR screens. Proc Natl Acad Sci U S A 115(50):25186-25195.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31727847">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Fei_PNAS_2019.pdf">[Full Paper]</a></li>
  <li>Zhang J, Hu X, Wang J, Sahu AD, Cohen D, Song L, Ouyang Z, Fan J, Wang B, Fu J, Gu S, Sade-Feldman M, Hacohen N, Li W, Ying X*, Li B*, <strong>Liu XS</strong>*. 2019. Immune receptor repertoires in pediatric and adult acute myeloid leukemia. Genome Med 11(1)73.<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31771646">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Zhang_GenMed_2019.pdf">[Full Paper]</a></li>
  <li>Li S#, Wan C#, Zheng R, Fan J, Dong X, Meyer CA*, <strong>Liu XS</strong>*. 2019. Cistrome-GO: a web server for functional enrichment analysis of transcription factor ChIP-seq peaks. Nucleic Acids Res 47(W1):W206-W211<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31053864">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Li_OxfordPress_2019.pdf">[Full Paper]</a></li>
  <li>Köster J, Brown M, <strong>Liu, XS</strong>. 2019. A Bayesian model for single cell transcript expression analysis on MERFISH data. Bioinformatics 35(6):995–1001.
<br><a href="https://www.ncbi.nlm.nih.gov/pubmed/30875429">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Koster_OxfordPress_2019.pdf">[Full Paper]</a></li>
  <li>Wang B#, Wang M#, Zhang W#, Xiao T, Chen CH, Wu A, Wu F, Traugh N, Wang X, Li Z, Mei S, Cui Y, Shi S, Lipp JJ, Hinterndorfer M, Zuber J, Brown M, Li W*, <strong>Liu XS</strong>*. 2019. Integrative analysis of pooled CRISPR genetic screens using MAGeCKFlute. Nat Protoc 14(3):756-780.
<br><a href="https://www.ncbi.nlm.nih.gov/pubmed/30710114">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Wang_NatPro_2019.pdf">[Full Paper]</a></li>
  <li>Hu X#, Zhang J#, Wang J, Fu J, Li T, Zheng X, Wang B, Gu S, Jiang P, Fan J, Ying X, Zhang J, Carroll MC, Wucherpfennig KW, Hacohen N, Zhang F, Zhang P, Liu JS*, Li B*, <strong>Liu XS</strong>*. 2019. Landscape of B cell immunity and related immune evasion in human cancers. Nat Genet 51(3):560-567. 
<br><a href="https://www.ncbi.nlm.nih.gov/pubmed/30742113">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Hu_NatGen_2018.pdf">[Full Paper]</a></li>
  <li>Campbell PT, Ambrosone CB, Nishihara R, Aerts HJWL, Bondy M, Chatterjee N, Garcia-Closas M, Giannakis M, Golden JA, Heng YJ, Kip NS, Koshiol J, <strong>Liu XS</strong>, Lopes-Ramos CM, Mucci LA, Nowak JA, Phipps AI, Quackenbush J, Schoen RE, Sholl LM, Tamimi RM, Wang M, Weijenberg MP, Wu CJ, Wu K, Yao S, Yu KH, Zhang X, Rebbeck TR, Ogino S. 2019. Proceedings of the fourth international molecular pathological epidemiology (MPE) meeting. Cancer Causes Control 30(8):799-811<br>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31069578">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Campbell_CCC_2019.pdf">[Full Paper]</a></li>
  <li>Cato L, de Tribolet-Hardy J, Lee I, Rottenberg JT, Coleman I, Melchers D, Houtman R, Xiao T, Li W, Uo T, Sun S, Kuznik NC, Göppert B, Ozgun F, van Royen ME, Houtsmuller AB, Vadhi R, Rao PK, Li L, Balk SP, Den RB, Trock BJ, Karnes RJ, Jenkins RB, Klein EA, Davicioni E, Gruhl FJ, Long HW, <strong>Liu XS</strong>, Cato ACB, Lack NA, Nelson PS, Plymate SR, Groner AC, Brown M. 2019. ARv7 Represses Tumor-Suppressor Genes in Castration-Resistant Prostate Cancer. Cancer Cell 35(3):401-413. e6
<br><a href="https://www.ncbi.nlm.nih.gov/pubmed/30773341">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Cato_CellPress_2019.pdf">[Full Paper]</a></li>
  <li>Rafiei S, Gui B, Wu J, <strong>Liu XS</strong>, Kibel AS, Jia L. 2019. Targeting the MIF/CXCR7/AKT Signaling Pathway in Castration-Resistant Prostate Cancer. Mol Cancer Res 17(1):263-276.
<br><a href="https://www.ncbi.nlm.nih.gov/pubmed/30224544">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Rafei_AACR_2018.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2018">2018</h2>
<ul>
  <li>Zheng R#, Wan C#, Mei S, Qin Q, Wu Q, Sun H, Chen CH, Brown M, Zhang X, Meyer CA, <strong>Liu XS</strong>. 2018. Cistrome Data Browser: expanded datasets and new tools for gene regulatory analysis. Nucleic Acids Res 47(D1):D729-D735.
<br><a href="https://www.ncbi.nlm.nih.gov/pubmed/30462313">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Zheng_Oxford_2018.pdf">[Full Paper]</a></li>
  <li>Chen CH#, Xiao T#, Xu H, Jiang P, Meyer CA, Li W*, Brown M*, <strong>Liu XS</strong>*. 2018. Improved design and analysis of CRISPR knockout screens. Bioinformatics 34(23):4095-4101 
<br><a href="https://www.ncbi.nlm.nih.gov/pubmed/29868757">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Chen_BioInfo_2018.pdf">[Full Paper]</a></li>
  <li>Hu X#, Zhang J#, Liu JS, Li B, <strong>Liu XS</strong>*. 2018. Evaluation of immune repertoire inference methods from RNA-seq data. Nat Biotechnol 36(11):1034.
<br><a href="https://www.ncbi.nlm.nih.gov/pubmed/30412202">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Hu_NatBio_2018.pdf">[Full Paper]</a></li>
  <li>Jiang P#, Gu S#, Pan D#, Fu J, Sahu A, Hu X, Li Z, Traugh N, Bu X, Li B, Liu J, Freeman GJ, Brown MA, Wucherpfennig KW*, <strong>Liu XS</strong>*. 2018. Signatures of T cell dysfunction and exclusion predict cancer immunotherapy response. Nat Med 24(10):1550-1558
<br><a href="https://www.ncbi.nlm.nih.gov/pubmed/30127393">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Jiang_NatMed_2018.pdf">[Full Paper]</a></li>
  <li>Xiao T#, Li W#, Wang X, Xu H, Yang J, Wu Q, Huang Y, Geradts J, Jiang P, Fei T, Chi D, Zang C, Liao Q, Rennhack J, Andrechek E, Li N, Detre S, Dowsett M, Jeselsohn RM, <strong>Liu XS</strong>*, Brown M*. 2018. Estrogen-regulated feedback loop limits the efficacy of estrogen receptor-targeted breast cancer therapy. Proc Natl Acad Sci U S A 115(31):7869-7878. 
<br><a href="https://www.ncbi.nlm.nih.gov/pubmed/29987050">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Xiao_ProcNat_2018.pdf">[Full Paper]</a></li>
  <li>Jiang P, Sellers W, <strong>Liu, XS</strong>. 2018. Big Data Approaches for Modeling Response and Resistance to Cancer Drugs. Annual Review of Biomedical Data Science 1:1-27.
<br><a href="https://www.annualreviews.org/doi/10.1146/annurev-biodatasci-080917-013350#abstractSection">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/big_data_approaches_for_modeling_response_and_resistance_to_cancer_drugs.pdf">[Full Paper]</a></li>
  <li>Jiang P, Lee W, Li X, Johnson C, Liu JS, Brown M, Aster JC*, <strong>Liu XS</strong>*. 2018. Genome-scale signatures of gene interaction from compound screens predict clinical efficacy of targeted cancer therapies. Cell Syst 6(3):343-354.
<br><a href="https://www.ncbi.nlm.nih.gov/pubmed/29428415">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Jiang_OCP_2018.pdf">[Full Paper]</a></li>
  <li>Pan D#, Kobayashi A#, Jiang P#, Ferrari de Andrade L, Tay RE, Luoma A, Tsoucas D, Qiu X, Lim K, Rao P, Long HW, Yuan GC, Doench J, Brown M, <strong>Liu XS</strong>*, Wucherpfennig KW*. 2018. A major chromatin regulator determines resistance of tumor cells to T cell-mediated killing. Science 359(6377):770-5.
<br><a href="https://www.ncbi.nlm.nih.gov/pubmed/29301958">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Pan_Science_2018.pdf">[Full Paper]</a>
<a href="https://drive.google.com/file/d/124h_Qr8T2xIBX4jAzeeU33f7oBcgZVIw/view?usp=sharing">[CRISPR screen data]</a></li>
  <li>Cader FZ, Schackmann RCJ, Hu X, Wienand K, Redd R, Chapuy B, Quyang J, Paul N, Gjini E, Lipshitz M, Armand P, Wu D, Fromm JR, Neuberg D, <strong>Liu XS</strong>, Rodig SJ, Shipp MA. 2018. Mass cytometry of Hodgkin lymphoma reveals a CD4+ exhausted T-effector and T-regulatory cell rich microenvironment. Blood 132(8):825-836
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/29880615">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Cader_BloodJournal_2018.pdf">[Full Paper]</a></li>
  <li>Cornwell M#, Vangala M#, Taing L#, Herbert Z, Koster J, Li B, Sun H, Li T, Zhang J, Qiu X, Pun M, Jeselsohn R, Brown M, <strong>Liu XS</strong>, Long HW#. 2018. VIPER: Visualization pipeline for RNA-seq, a Snakemake workflow for efficient and complete RNA-seq analysis. BMC Bioinformatics 19(1):135.
<br><a href="https://www.ncbi.nlm.nih.gov/pubmed/29649993">[Abstract]</a>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5897949/">[Full Paper]</a></li>
  <li>Sun T, Du SY, Armenia J, Qu F, Fan J, Wang X, Fei T, Komura K. <strong>Liu XS</strong>, Lee GSM, Kantoff PW. 2018. Expression of IncRNA MIR222HG co-transcribed from the miR-221/222 gene promoter facilitates the development of castration-resistant prostate cancer. Oncogenesis 7(3):30.
<br><a href="https://www.ncbi.nlm.nih.gov/pubmed/29540675">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Sun_NPG_2018.pdf">[Full Paper]</a></li>
  <li>Jeselsohn R, Bergholz JS, Pun M, Cornwell M, Liu W, Nardone A, Xiao T, Li W, Qiu X, Buchwalter G, Feiglin A, Abell-Hart K, Fei T, Rao P, Long H, Kwiatkowski N, Zhang T, Gray N, Melchers D, Houtman R, <strong>Liu XS</strong>, Cohen O, Wagle N, WIner EP, Zhao J, Brown M. 2018. Allele-specific chromatin recruitment and therapeutic vulnerabilities of ESR1 activating mutations. Cancer Cell 33(2):173-186.
<br><a href="https://www.ncbi.nlm.nih.gov/pubmed/29438694">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Jeselsohn_CellPress_2018.pdf">[Full Paper]</a></li>
  <li>Chen D, Fang L, Mei S, Li H, Xu X, Des Marais TL, Lu K, <strong>Liu XS</strong>, Jin C. 2018. Erratum: Regulation of chromatin assembly and cell transformation by formaldehyde exposure in human cells. Environ Health Perspect 126(1):019001.
<br><a href="https://www.ncbi.nlm.nih.gov/pubmed/29326091">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Chen_EHP_2018.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2017">2017</h2>
<ul>
  <li>Li T, Fan J, Wang B, Traugh N, Chen Q, Liu S, Li B*, <strong>Liu XS</strong>*. 2017. TIMER: A web server for comprehensive analysis of tumor-infiltrating immune cells. Cancer Res 77(21):e108-110. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/29092952">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CancerRes17_e108.pdf">[Full Paper]</a></li>
  <li>Mei S#, Meyer CA#, Zheng R, Qin Q, Wu Q, Jiang P, Li B, Shi X, Wang B, Fan J, Brown M, Zang C*, <strong>Liu XS</strong>*. 2017. Cistrome Cancer: a web resource for integrative gene regulation modeling in cancer. Cancer Res 77(21):e19-22. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/29092931">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CancerRes17_e19.pdf">[Full Paper]</a></li>
  <li>Cao Q, Ma J, Chen C-H, Xu H, Chen Z*, Li W*, <strong>Liu XS</strong>*. 2017. CRISPR-FOCUS: a web server for designing focused CRISPR screening experiments. PLoS One 12(9):e0184281.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/28873439">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PLosOne17_184281.pdf">[Full Paper]</a></li>
  <li>Li B, Liu JS*, <strong>Liu XS</strong>*. 2017. Revisit linear regression-based deconvolution methods for tumor gene expression data. Genome Biology 18(1):127.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/28679386">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenomeBiol17_127.pdf">[Full Paper]</a></li>
  <li>Li MJ, Zhang J, Liang Q, Xuan C, Wu J, Jiang P, Li W, Zhu Y, Wang P, Fernandez D, Shen Y, Chen Y, Kocher JA, Yu Y, Sham PC, Wang J*, Wang Liu JS*, <strong>Liu XS</strong>*. 2017. Exploring genetic associations with ceRNA regulation in the human genome. Nucleic Acids Res 45(10):5653-5665.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/28472449">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NucleicAcidsRes17_May2.pdf">[Full Paper]</a></li>
  <li>Liu Y#, Chen S#, Wang S, Soares F, Fischer M, Meng F, Du Z, Lin C, Meyer C, DeCaprio JA, Brown M*, <strong>Liu XS</strong>*, He HH. 2017. Transcriptional landscape of the human cell cycle. PNAS USA 114(113):3473-3478.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/28289232">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PNAS17_Mar13.pdf">[Full Paper]</a></li>
  <li><strong>Liu XS</strong>*, Mardis ER*. 2017. Applications of immunogenomics to cancer. Cell 168(4):600-612. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/28187283">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Cell17_600.pdf">[Full Paper]</a></li>
  <li>Li B*, Li T, Wang B, Dou R, Zhang J, Liu JS, <strong>Liu XS</strong>*. 2017. Ultrasensitive detection of TCR hypervariable-region sequences in solid-tissue RNA-seq data. Nat Genet 49(4):482-483.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/28358132">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatGenet17_482.pdf">[Full Paper]</a></li>
  <li>Mei S#, Qin Q#, Wu Q#, Sun H, Zhen R, Zang C, Zhu M, Wu J, Shi X, Taing L, Liu T, Brown M, Meyer CA*, <strong>Liu XS</strong>*. 2017. Cistrome data browser: a data portal for ChIP-Seq and chromatin accessibility data in human and mouse. Nucleic Acids Res 45(D1):D658-D662. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/27789702">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NucleicAcidsRes16_Oct26.pdf">[Full Paper]</a></li>
  <li>Chen D, Fang L, Mei S, Li H, Xu X, Des Marais TL, Lu K, <strong>Liu XS</strong>, Jin C. 2017. Regulation of chromatin assembly and cell transformation by formaldehyde exposure in human cells. Environ Health Perspect 125(9):097019. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/28937961">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/EHP17_097019.pdf">[Full Paper]</a></li>
  <li>Gao W, Li W, Xiao T, <strong>Liu XS</strong>, Kaelin WG Jr. 2017. Inactivation of the PBRM1 tumor suppressor gene amplifies the HIF-response in VHL-/- clear cell renal carcinoma.Proc Natl Acad Sci U S A 114(5):1027-1032. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/28082722">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PNAS17_1027.pdf">[Full Paper]</a></li>
  <li>Choi SH, Severson E, Pear WS, <strong>Liu XS</strong>, Aster JC, Blacklow SC. 2017. The common oncogenomic program of NOTCH1 and NOTCH3 signaling in T-cell acute lymphoblastic leukemia. PLoSOne 12(10):e0185762.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/29023469">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PLosOne17_185762.pdf">[Full Paper]</a></li>
  <li>Severson E, Arnett KL, Wang H, Zang C Taing L, Liu H, Pear WS, <strong>Liu XS</strong>, Blacklow SC, Aster JC. 2017. Genome-wide identification and characterization of Notch transcription complex-binding sequence-paired sites in leukemia cells. SciSignal 10(477).pil:eaag1598.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/28465412">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/SciSignal17_eaag.pdf">[Full Paper]</a></li>
  <li>Carey CD, Gusenleitner D, Lipschitz M, Roemer MGM, Stack EC, Gjini E, Hu X, Redd R, Freeman GJ, Neuberg D, Hodi FS, <strong>Liu XS</strong>, Shipp MA, Rodig SJ. 2017. Topological analysis reveals a PD-L1 associated microenvironmental niche for Reed-Sternberg cells in Hodgkin Lymphoma. Blood 130(22):2420-2430. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/28893733">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Blood17_Sept11.pdf">[Full Paper]</a></li>
  <li>Fei T, Chen Y, Xiao T, Li W, Cato L, Zhang P, Cotter MB, Bowden M, Lis RT, Zhao SG, Wu Q, Feng FY, Loda M, He HH, <strong>Liu XS</strong> Brown M. 2017. Genome-wide CRISPR screen idetifies HNRNPL as a prostate cancer dependecy regulating RNA splicing. PNAS 114(26):E5207-E5215.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/28611215">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PNAS17_E5207.pdf">[Full Paper]</a></li>
  <li>Huang T#, Zhang P#, Li W#, Zhao T, Zhang Z, Chen S, Yang Y, Feng Y, Li F, <strong>Liu XS</strong>, Jiang G, Zhang F*. 2017. G91 promotes tumor cell growth and invasion by silencing CASP1 in non-small-cell lung cancer cells. Cell Death Dis 8(4):e2726.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/28383547">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CellDeathDis17_e2726.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2016">2016</h2>
<ul>
  <li>Wang S#, Zang C#, Xiao T, Fan J, Mei S, Qin Q, Wu Q , Li XJ, Xu K, He HH, Brown M, Meyer CA*, <strong>Liu XS</strong>*. 2016. Modeling cis-regulation with a compendium of genome-wide histone H3K27ac profiles. Genome Res 26(10)1417-1429. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/27466232">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenomeRes16_Jul27.pdf">[Full Paper]</a></li>
  <li>Ma J, Koster J, Qin Q, Hu S, Li W, Chen C, Cao Q, Wang J, Mei S, Liu Q*, Xu H*, <strong>Liu XS</strong>*. 2016. CRISPR-DO for genome-wide CRISPR design and optimization.Bioinformatics 32(21):3336-38. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/27402906">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Bioinformatics16_32.pdf">[Full Paper]</a></li>
  <li>Zhu S#, Li W#, Liu J, Chen CH Liao Q, Xu P, Xu H, Xiao T, Cao Z, Peng J, Yuan P, Brown M, <strong>Liu XS</strong>*, Wei W*. 2016. Genome-scale deletion screening of human long non-coding RNAs using a paired-guide RNA CRISPR-Cas9 library. Nature Biotech 34(12):1279-86. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/27798563">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatBiotech16_1279.pdf">[Full Paper]</a></li>
  <li>Qin Q#, Mei S#, Wu Q#, Sun H, Li L, Taing L, Chen S, Li F, Liu T, Zang C, Xu H, Chen Y, Meyer CA, Zhang Y, Brown M, Long HW*, <strong>Liu XS</strong>*. 2016. ChiLin: a comprehensive ChIP-seq and DNase -seq quality control and analysis pipeline. BMC Bioinformatics 17(1):404. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/27716038">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/BMCBioinformatics16_404.pdf">[Full Paper]</a></li>
  <li>Zang C#, Luyten A#, Chen J, <strong>Liu XS</strong>, Shivdasani RA*. 2016. NF-E2, FLI1 and RUNX1 collaborate at areas of dynamic chromatin to activate transcription in mature mouse megakaryocytes. Sci Rep 6:30255. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/27457419">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/srep16_30255.pdf">[Full Paper]</a></li>
  <li>Li B, Severson E, Pignon JC, Zhao H, Li T, Novak J, Jiang P, Shen H, Aster JC, Rodig S, Signoretti S, Liu JS*, <strong>Liu XS</strong>*. 2016. Comprehensive analyses of tumor immunity: implications for cancer immunotherapy. Genome Biol 17(1):174. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/27549193">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenomeBiol16_174.pdf">[Full Paper]</a>
<a href="http://cistrome.org/TIMER/download.html">[Data Download]</a></li>
  <li>Li B#, Li T#, Pignon JC, Wang B, Wang J, Shukla SA, Dou R, Chen Q, Hodi FS, Choueiri TK, Wu C, Hacohen N, Signoretti S, Liu JS*, <strong>Liu XS</strong>*. 2016. Landscape of tumor-infiltrating T cell repertoire of human cancers. Nat Genet 48(7):725-32. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/27240091">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatGenet16_725.pdf">[Full Paper]</a></li>
  <li>Zang C#, Wang T#, Deng K, Li B, Hu S, Qin Q, Xiao T, Zhang S, <strong>Meyer CA</strong>, He HH, Brown M, Liu JS, Xie Y*, <strong>Liu XS</strong>*. 2016. High-dimensional genomic data bias correction and data integration using MANCIE. Nat Commun 7:11305. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/27072482">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatCommun16_11305.pdf">[Full Paper]</a></li>
  <li>Du Z#, Sun T#, Hacisuleyman E, Fei T, Wang X, Brown M, Rinn JL, Lee MG, Chen Y*, Kantoff PW*, <strong>Liu XS</strong>*. 2016. Integrative analyses reveal a long noncoding RNA-mediated sponge regulatory network in prostate cancer. Nat Commun 7:10982. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/26975529">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatCommun16_10982.pdf">[Full Paper]</a></li>
  <li>Liu Y, Fei T, Zheng X, Brown M, Zhang P*, <strong>Liu XS</strong>*, Wang H*. 2016. An integrative pharmacogenomic approach identifies two-drug combination therapies for personalized cancer medicine. Sci Rep 6:22120. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/26916442">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/SciRep16_22120.pdf">[Full Paper]</a></li>
  <li>Xu H#, Xu K#, He HH, Zang C, Chen CH, Chen Y, Qin Q, Wang S, Hu S, Li F, Long H, Brown M*, <strong>Liu XS</strong>*. 2016. Integrative analysis reveals the transcriptional collaboration between EZH2 and E2F1 in the regulation of cancer-related gene expression. Mol Cancer Res 14(2):163-72. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/26659825">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/MolCancerRes16_163.pdf">[Full Paper]</a></li>
  <li>Lu T, Aron L, Zullo J, Pan Y, Kim H, Chen Y, Yang TH, Kim HM, Drake D, <strong>Liu XS</strong>, Bennett DA, Colaiacovo MP, Yankner BA*. 2016. Addendum: REST and stress resistance in ageing and Alzheimer's disease. Nature 540(7633):470. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/27851730">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Nature16_Addendumto14_448.pdf">[Corrections and Amendments]</a></li>
  <li>Ni T#, Li XY#, Lu N#, An T, Liu Zp, Fu R, Zhang YW, Xu XJ, Grant Rowe R, Lin YS, Scherer A, Feinberg T, Zhen XQ, Chen BA, <strong>Liu XS</strong>, Guo QL, Wu ZQ*, Weiss SJ*. 2016. Snail1-dependent p53 repression regulates expansion and activity of tumour-initiating cells in breast cancer. Nat Cell Biol 18(11):1221-1232.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/27749822">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatureCellBiol16_1221.pdf">[Full Paper]</a></li>
  <li>Cejas P, Li L, O'Neill NK, Duarte M, Rao P, Bowden M, Zhou CW, Mendiola M, Burgos E, Feliu J, Moreno-Rubio J, Guadalajara H, Moreno V, Garcia-Olmo D, Bellmunt J, Mullane S, Hirsch M, Sweeney CJ, Richardson A, <strong>Liu XS</strong>, Brown M, Shivdasani RA*, Long HW*. 2016. Chromatin immunoprecipitation from fixed clinical tissues reveals tumor-specific enhancer profiles. Nat Med 22(6):685-91. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/27111282">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatMed16_685.pdf">[Full Paper]</a></li>
  <li>Kong L#, Zhang P#, Li W, Yang Y, Tian Y, Wang X, Chen S, Yang Y, Huang T, Zhao T, Tang L, Su B, Li F, <strong>Liu XS</strong>, Zhang F*. 2016. KDM1A promotes tumor cell invasion by silencing TIMP3 in non-small cell lung cancer cells. Oncotarget 7(19):27959-74. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/27058897">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Oncotarget16_27959.pdf">[Full Paper]</a></li>
  <li>Goel S*, Wang Q, Watt AC, Tolaney SM, Dillon DA, Li W, Ramm S, Palmer AC, Yuzugullu H, Varadan V, Tuck D, Harris LN, Wong KK, <strong>Liu XS</strong>, Sicinski P, Winer EP, Krop IE, Zhao JJ*. 2016. Overcoming therapeutic resistance in HER2-positive breast cancers with CDK4/6 inhibitors. Cancer Cell 29(3):255-269. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/26977878">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CancerCell16_255.pdf">[Full Paper]</a></li>
  <li>Shu S#, Lin CY#, He HH#, Witwicki RM#, Tabassum DP, Roberts JM, Janiszewska M, Huh SJ, Liang Y, Ryan J, Doherty E, Mohammed H, Guo H, Stover DG, Ekram MB, Peluffo G, Brown J, D'Santos C, Krop IE, Dillong D, McKeown M, Ott C, Qi J, Ni M, Rao PK, Duarte M, Wu SY, Chiang CM, Anders L, Young RA, Winer EP, Letai A, Barry WT, Carroll JS, Long HW, Brown M, <strong>Liu XS</strong>, Meyer CA, Bradner JE*, Polyak K*. 2016. Response and resistance to BET bromodomain inhibitors in triple-negative breast cancer. Nature 529(7586):413-7. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/26735014">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Nature16_413.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2015">2015</h2>
<ul>
  <li>Li W#, Koster J#, Xu H, Chen CH, Xiao T, Liu JS, Brown M*, <strong>Liu XS</strong>*. 2015. Quality control, modeling and visualization of CRISPR screens with MAGeCK-VISPR. Genome Biol 16(1):281.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/26673418">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenomeBiol15_281.pdf">[Full Paper]</a></li>
  <li>Jiang P, Wang H, Li W, Zang C, Li B, Wong YJ, Meyer C, Liu JS, Aster JC, <strong>Liu XS</strong>*. 2015. Network analysis of gene essentiality in functional genomics experiments. Genome Biol 16:239.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/26518695">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenomeBiol15_239.pdf">[Full Paper]</a></li>
  <li>Zhang N#, Wang H#, Fang Y, Wang J*, Zheng X*, <strong>Liu XS</strong>*. 2015. Predicting anticancer drug responses using a dual-layer integrated cell line-drug network model. PLOS Computational Biology 11(9):e1004498.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/26418249">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PLoSComputationalBiology15_1004498.pdf">[Full Paper]</a></li>
  <li>Xu H#, Xiao T#, Chen CH, Meyer CA, Wu Q, Wu D, Cong L, Zhang F, Liu JS, Brown M*, <strong>Liu XS</strong>*. 2015. Sequence determinants of improved CRISPR sgRNA design. Genome Res 25:(8)1147-57.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/26063738">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenomeRes15_1147.pdf">[Full Paper]</a></li>
  <li>Jiang P, Freedman ML, Liu JS, <strong>Liu XS</strong>*. 2015. Inference of transcriptional regulation in cancers. PNAS 112(25):7731-6.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/26056275">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/pnas15_doi_10.1073.pdf">[Full Paper]</a></li>
  <li>Jiang P, <strong>Liu XS</strong>*. 2015. Big data mining yields novel insights on cancer. Nat Genet 47(2):103-4.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/25627899">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatGen15_103.pdf">[Full Paper]</a></li>
  <li>Du Z, Fei T, Brown M, <strong>Liu XS</strong>, Chen Y. Data integration on noncoding RNA Studies. 2015. Chapter in: Tseng G, Ghosh D, Zhou XJ, eds. Integrating Omics Data.Cambridge University Press: Cambridge, England. 403-424. 
<br><a href="https://liulab-dfci.github.io/resources/publications/2015_IntegratingOmicsData_Chapter.pdf">[Full Chapter]</a></li>
  <li>Zhang J#, Wang C#, Chen X, Takada M, Fan C, Zheng X, Wen H, Liu Y, Wang C, Pestell RG, Aird KM, Kaelin WG Jr <strong>Liu XS</strong>, Zhang Q*. 2015. EgIN2 associates with NFR-1PGC1a omplex and controls mitochondrial function in breast cancer. EMBO J 34(23):2953-70.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/26492917">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/EMBO15_Octepub.pdf">[Full Paper]</a></li>
  <li>Fei Q#, Yang X#, Jiang H, Wang Q, Yu Y, Yu Y, Yi W, Zhou S, Chen T, Lu C, Atadja P, <strong>Liu XS</strong>, Li E, Zhang Y*, Shou J*. 2015. SETDB1 modulates PRC2 activity at developmental genes independently of H3K9 trimethylation in mouse ES cells. Genome Res 25(9):1325-35.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/26160163">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenomeRes15_Jul9.pdf">[Full Paper]</a></li>
  <li>Wang H, Zang C, <strong>Liu XS</strong>, Aster JC*. 2015. The role of notch receptors in transcriptional regulation. J Cell Physiol 230(5):982-8.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/25418913">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/JCellPhysiol15_982.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2014">2014</h2>
<ul>
  <li>Zhang Y#,*, Vastenhouw NL#,*, Feng J, Fu K, Wang C, Ge Y, van Hummelen P, Schier AF*, <strong>Liu XS</strong>*. 2014. Canonical nucleosome organization at promoters forms during genome activation. Genome Res 24(2):260-6. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/24285721">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenomeRes14_260.pdf">[Full Paper]</a></li>
  <li>Wang Q#, Huang J#, Sun H#, Liu J, Wang J, Wang Q, Qin Q, Mei S, Zhao C, Yang X, <strong>Liu XS</strong>*, Zhang Y*. 2014. CR Cistrome: a ChIP-Seq database for chromatin regulators and histone modification linkages in human and mouse. Nucl Acids Res 42:D450-8.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/24253304">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NucleicAcidsRes14_D450.pdf">[Full Paper]</a></li>
  <li>Wang H#, Zang C#, Taing L, Arnett K, Wong YJ, Pear WS, Blacklow SC, <strong>Liu XS</strong>*, Aster JC*. 2014. NOTCH1/RBPJ complexes drive target gene expression through dynamic interactions with super-enhancers. PNAS 111(2):705-10. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/24374627">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PNAS14_705.pdf">[Full Paper]</a></li>
  <li>Zheng X#, Zhao Q#, Wu HJ#, Li W, Wang H, Meyer CA, Qin QA, Xu H, Zang C, Jiang P, Li F, Hou Y, He J, Wang J, Wang J, Zhang P*, Zhang Y*, <strong>Liu XS</strong>*. 2014. MethylPurify:tumor purity deconvolution and differential methylation detecton from single tumor DNA methylomes. Genome Biol 15(8):419.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/25103624">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenomeBiol14_419.pdf">[Full Paper]</a></li>
  <li>Meyer CA*, <strong>Liu XS</strong>*. 2014. Identifying and mitigating bias in next-generation sequencing methods for chromatin biology. Nat Rev Genet 15(11):709-21.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/25223782">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatRevGenet14_709.pdf">[Full Paper]</a></li>
  <li>Luyten A#, Zang C#, <strong>Liu XS</strong>*, Shivdasani RA*. 2014. Active enhancers are delineated de novo during hematopoiesis, with limited lineage fidelity among specified primary blood cells. Genes Dev 2014; 28(16):1827-39. Genes Dev 28(16):1827-39.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/25128499">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenesDev14_1827.pdf">[Full Paper]</a></li>
  <li>Meng FL#, Du Z#, Federation A, Hu J, Wang Q, Kieffer-Kwon KR, Meyers RM, Amor C, Wasserman CR, Neuberg D, Casellas R, Nussenzweig MC, Bradner JE*, <strong>Liu XS</strong>*, Alt FW*. 2014. Convergent transcription at intragenic super-enhancers targets AID-initiated genomic instability. Cell 159(7):1538-48.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/25483776">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Cell14_1538.pdf">[Full Paper]</a></li>
  <li>Li W#, Xu H#, Xiao T, Cong L, Love MI, Zhang F, Irizarry RA, Liu JS, Brown M, <strong>Liu XS</strong>*. 2014. MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens. Genome Biol 15(12):554.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/25476604">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenomeBiol14_554.pdf">[Full Paper]</a></li>
  <li>Ho JW#, Jung YL#, Liu T#, et al, Strome S*, Elgin SC*, <strong>Liu XS</strong>*, Lieb JD*, Ahringer J*, Karpen GH*, Park PJ*. 2014. Comparative analysis of metazoan chromatin organization. Nature 512(7515):449-52. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/25164756">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Nature14_449.pdf">[Full Paper]</a></li>
  <li>He HH#, Meyer CA#, Hu SS#, Chen MW, Zang C, Liu Y, Rao PK, Fei T, Xu H, Long H, <strong>Liu XS</strong>*, Brown M*. 2014. Refined DNase-seq protocol and data analysis reveals intrinsic bias in transcription factor footprint identification. Nat Methods 11(1):73-8. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/24317252">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatMethods13_Dec8.pdf">[Full Paper]</a></li>
  <li>Sun T, Wang X, He HH, Sweeney CJ, <strong>Liu XS</strong>, Brown M, Balk S, Lee GS*, Kantoff PW*. 2014. MiR-221 promotes the development of androgen independence in prostate cancer cells via downregulation of HECTD2 and RAB1A. Oncogene 33(21):2790-80.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/23770851">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Oncogene14_2790.pdf">[Full Paper]</a></li>
  <li>Wee S, Dhanak D, Li H, Armstrong SC, Copeland RA, Sims R,Baylin SB, <strong>Liu XS</strong>, Schweizer L*. 2014. Targeting epigenetic regulators for cancer therapy. Annals NYAS 1309:30-6.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/24571255">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NYAS14_30.pdf">[Full Paper]</a></li>
  <li>Watanabe H#, Ma Q#, Peng S, Adelmant G, Swain D, Song W, Fox C, Francis JM, Pedamallu CS, Deluca DS, Brooks AN, Wang S, Que J, Rustgi AK, Wong KK, Ligon KL, <strong>Liu XS</strong>, Marto JA, Meyerson M, Bass AJ*. 2014. SOX2 and p63 colocalize at genetic loci in squamous cell carcinomas. J Clin Invest 124(4):1636-45.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/24590290">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/JClinInvest14_Mar3.pdf">[Full Paper]</a></li>
  <li>Chen X, Iliopoulos D, Zhang Q, Tang Q, Greeenblatt MB, Hatziapostolou M, Ni M, Chen Y, Lim E, Hu DZ, Hu B, Song M, Brown M, <strong>Liu XS</strong>, Glimcher LH*. 2014. XBP1 sustains tumor initiating cells in triple negative breast cancer by controlling the hypoxia response. Nature 508(7494):103-7.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/24670641">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Nature14_103.pdf">[Full Paper]</a></li>
  <li>Lu T, Aron L, Zullo J, Pan Y, Kim H, Chen Y, Tang TH, Kim HM, Drake D, <strong>Liu XS</strong>, Bennett DA, Colaiacovo MP, Yankner BA*. 2014. REST and stress resistance in ageing and Alzheimer's disease. Nature 507(7493):448-54.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/24670762">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Nature14_448.pdf">[Full Paper]</a></li>
  <li>Yamamoto S#, Wu Z#, Russnes HG, Takagi S, Peluffo G, Vaske C, Zhao X, Moen Vollan HK, Maruyama R, Ekram MB, Sun H, Kim JH, Carver K, Zucca M, Feng J, Almendro V, Bessarabova M, Rueda OM, Nikolsky Y, Caldas C, <strong>Liu XS</strong>, Polyak K*. 2014. JARID1B is a luminal lineage-driving oncogene in breast cancer. Cancer Cell 25(6):762-77.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/24937458">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CancerCell14_762.pdf">[Full Paper]</a></li>
  <li>Hsieh CL, Fei T, Chen Y, Li T, Gao Y, Wang X, Sun T, Sweeney CJ, Lee GS, Chen S, Balk SP, <strong>Liu XS</strong>, Brown M, Kantoff PW*. 2014. Enchancer RNAs participate in androgen receptor-driven looping that selectively enhances gene activation. Proc Natl Acad Sci USA 111(20):7319-24.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/24778216">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PNAS14_7319.pdf">[Full Paper]</a></li>
  <li>Wang J, Lin X, Wang S, Wang C, Wang Q, Duan X, Lu P, Wang Q, Wang C, <strong>Liu XS</strong>, Huang J*. 2014. PHF8 and REST/NRSF co-occupy gene promoters to regulate proximal gene expression. Sci Rep 4:5508.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/24852203">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/SciRep14_5008.pdf">[Full Paper]</a></li>
  <li>Stoeck A*, Lejnine S, Truong A, Pan L, Wang H, Zang C, Yuan J, Ware C, MacLean J, Garrett-Engele PW, Kluk M, Laskey J, Haines BB, Moskaluk C, Zawel L, Fawell S, Gilliland G, Zhang T, Kremer B, Knoechel B, Bernstein Be, Pear WS, <strong>Liu XS</strong>, Aster JC*, Sathyanarayanan S*. 2014. Discovery of biomarkers predictive of GSI response in triple negative breast cancer and adenoid cystic carcinoma. Cancer Discov 4(10):1154-67.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/25104330">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CancerDiscov14_1154.pdf">[Full Paper]</a></li>
  <li>Cai C#,*, He HH#,*, Gao S, Chen S, Yu Z, Gao Y, Chen S, Chen MW, Zhang J, Ahmed M, Wang Y, Metzger E, Schule R, <strong>Liu XS</strong>, Brown M, Balk SP. 2014. Lysine-specific demethylase 1 has dual functions as a major regulator of androgen receptor transcriptional activity. Cell Rep 9(5):1618-1627.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/25482560">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CellRep14_1618.pdf">[Full Paper]</a></li>
  <li>Yashiro-Ohtani Y#, Wang H#, Zang C, Arnett KL, Bailis W, Ho Y, Knoechel B, Lanauze C, Louis L, Forsyth KS, Chen S, Chung Y, Schug J, Blobel GA, Liebhaber SA, Bernstein BE, Blacklow SC, <strong>Liu XS</strong>, Aster JC, Pear WS*. 2014. Long-range enhance activity determines Myc sensitivity to Notch inhibitors in T cell leukemia. Proc Natl Acad Sci U S A 111(46):E4946-53.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/25369933">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PNAS14_4946.pdf">[Full Paper]</a></li>
  <li>Hu CY, Mohtat D, Yu Y, Ko YA, Shenoy N, Bhattacharya S, Izquierdo MC, Park AS, Giricz O, Vallumsetla N, Gundabolu K, Ware K, Bhagat TD, Suzuki M, Pullman J, <strong>Liu XS</strong>, Greally JM, Susztak K*, Verma A*. 2014. Kidney cancer is characterized by aberrant methylation of tissue-specific enhancers that are prognostic for overall survival. Clin Cancer Res 20(16):4349-60.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/24916699">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/ClinCancerRes14_4349.pdf">[Full Paper]</a></li>
  <li>Sulahian R, Casey F, Shen J, Qian ZR, Shin H, Ogino S, Weir BA, Vazquez F, <strong>Liu XS</strong>, Hahn WC, Bass AJ, Chan V, Shivdasani RA. 2014. An integrative analysis reveals functional targets of GATA6 transcriptional regulation in gastric cancer. Oncogene 33(49):5637-48
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/24317510">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Oncogene13_Dec9.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2013">2013</h2>
<ul>
  <li>Du Z#, Li H#, Wei Q#, Zhao X, Wang C, Xu W, <strong>Liu XS</strong>*, Jin W*, Su Z*. 2013. Genome-wide analysis of histone modifications: H3K4me2, H3K4me3, H3K9ac, and H3K27ac in oryza sativa L. japonica. Mol Plant 6(5):1463-72. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/23728290">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/MolPlant13_1463.pdf">[Full Paper]</a></li>
  <li>Du Z#, Fei T#, Verhaak RGW, Su Z, Zhang Y,Brown M*, Chen C*, <strong>Liu XS</strong>*. 2013. Integrative genomic analyses reveal clinically relevant long noncoding RNAs in human cancer. Nat Struct Mol Biol 20(7):908-13. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/23728290">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatStructMolBiol13_908.pdf">[Full Paper]</a></li>
  <li>Sun H, Qin B, Liu T, Wang Q, Liu J, Wang J, Lin X, Yang Y, Taing L, Rao PK, Brown M, Zhang Y, Long HW*, <strong>Liu XS</strong>*. 2013. CistromeFinder for ChIP-seq and DNase-seq data reuse. Bioinformatics 29(10):1352-4. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/23508969">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Bioinformatics13_1352.pdf">[Full Paper]</a></li>
  <li>Ni M#, Chen Y#, Fei T, Li D, Lim E, <strong>Liu XS</strong>*, Brown M*. 2013. Amplitude modulation of androgen signaling by c-MYC. Genes Dev 27(7):734-48. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/23530127">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenesDev13_734.pdf">[Full Paper]</a></li>
  <li>Shin H#, Liu T#, Duan X, Zhang Y*, <strong>Liu XS</strong>*. 2013. Computational methodology for ChIP-seq analysis. Quantitative Biology 1(1):54-70. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/25741452">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/QuantitativeBiol13_54.pdf">[Full Paper]</a></li>
  <li>Verzi MP#, Shin H#, San Roman AK, <strong>Liu XS</strong>*, Shivdasani RA*. 2013. Intestinal master transcription factor CDX2 controls chromatin access for partner transcription factor binding. Mol Cell Biol 33(2):281-92. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/23129810">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CellBiol13_281.pdf">[Full Paper]</a></li>
  <li>Wang S, Sun H, Ma J, Zang C, Wang C, Wang J, Tang Q, Meyer CA, Zhang Y, <strong>Liu XS</strong>. 2013. Target analysis by integration of transcriptome and ChIP-seq data with BETA. Nat Protoc 8(12):2502-15. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/24263090">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatProtoc13_2502.pdf">[Full Paper]</a></li>
  <li>Wang H*, Meyer CA, Fei T, Wang G, Zhang F*, <strong>Liu XS</strong>. 2013. A systematic approach identifies FOXA1 as a key factor in the loss of epithelial traits during the epithelial-to-mesenchymal transition in lung cancer. BMC Genomics 14:680.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/24093963">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/BMCGenomics13_680.pdf">[Full Paper]</a></li>
  <li>Catic A, Suh CY, Hill CT, Daheron L, Henkel T, Orford KW, Dombkowski DM, Liu T, <strong>Liu XS</strong>, Scadden DT. 2013. Genome-wide map of nuclear protein degradation shows NCoR1 turnover as a key to Mitochondrial gene regulation. Cell 155(6):1380-95. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/24315104">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Cell13_1380.pdf">[Full Paper]</a></li>
  <li>Trynka G, Sandor C, Han B, Xu H, Stranger BE, <strong>Liu XS</strong> Raychaudhuri S*. 2013. Chromatin marks identify critical cell types for fine mapping complex trait variants. Nat Genet 45(2):124-30. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/23263488">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatGenet13_124.pdf">[Full Paper]</a></li>
  <li>Zhang X, Peterson KA, <strong>Liu XS</strong>, McMahon AP, Ohba S. 2013. Gene regulatory networks mediating canonical Wnt signal directed control of pluripotency and differentiation in embryo stem cells. Stem Cells 31(12):2667-79. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/23505158">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/StemCells13_Mar15epub.pdf">[Full Paper]</a></li>
  <li>Cai C, Wang H, He HH, Chen S, He L, Ma F, Mucci L, Wang Q, Fiore C, Sowalsky AG, Loda M, <strong>Liu XS</strong>, Brown M, Balk SP*, Yuan X*. 2013. ERG induces androgen receptor-mediated regulation of SOX9 in prostate cancer. J Clin Invest 123(3):1109-22. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/23426182">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/JClinInvest13_1109.pdf">[Full Paper]</a></li>
  <li>Choudhury S#, Almendro V#, Merino VF#, Wu Z#, Maruyama R#, Su Y, Martins FC, Fackler MJ, Bessarabova M, Kowalczyk A, Consway T, Beresford-Smith B, MacIntyre G, Cheng YK, Lopez-Bujanda Z, Kaspi A, Hu R, Robens J, Nikolskaya T, Haakensen VD, Schnitt SJ, Argani P, Ethington G, Panos L, Grant M, Clark J, Herlihy W, Lin SJ, Chew G, Thompson EW, Greene-Colozzi A, Richardson AL, Rosson GD, Pike M, Garber JE, Nikolsky Y, Blum JL, Au A, Hwang ES, Tamimi RM, Michor F, Haviv I, <strong>Liu XS</strong>, Sukumar S*, Polyak K*. 2013. Molecular profiling of human mammary gland links breast cancer risk to a p27+ cell population with progenitor characteristics. Cell Stem Cell 13(1):117-30. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/23770079">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CSC13_117.pdf">[Full Paper]</a></li>
  <li>Tam WL, Lu H, Burkhuisen J, Soh BS, Lim E, Reinhardt F, Wu ZJ, Krall JA, Bierie B, Guo W, Chen X, <strong>Liu XS</strong>, Brown M, Lim B, Weinberg RA. 2013. Protein kinase C a is a central signaling node and therapeutic target for breast cancer stem cells. Cancer Cell 24(3):347-64. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/24029232">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CancerCell13_347.pdf">[Full Paper]</a></li>
  <li>He HH, Meyer CA, Long H, <strong>Liu XS</strong>, Brown M. 2013. A closer look into DNase I hypersensitivity. Epigenetics and Chromatin 6(Suppl 1):P25.
<br><a href="https://liulab-dfci.github.io/resources/publications/EpigenChroma6_1.pdf">[Full Paper]</a></li>
  <li>Xu K, Fei T, Wu ZJ, Su Z, Groner AC, He HH, Cai C, Stack EC, Loda M, Liu T, Morrissey C, Vessella RL, Kantoff PW, Balk SP, <strong>Liu XS</strong>, Brown M. 2013. Polycomb-independent activity of EZH2 in castration resistant prostate cancer. Epigenetics and Chromatin 6(Suppl 1):O14.
<br><a href="https://liulab-dfci.github.io/resources/publications/EpigeneticsChromatin13_O14.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2012">2012</h2>
<ul>
  <li>Wang C, Tian R, Zhao Q, Xu H, Meyer CA, Li C, Zhang Y, <strong>Liu XS</strong>. 2012. Computational inference of mRNA stability from histone modification and transcriptome profiles. Nucleic Acids Res 40(14):6414-23. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/22495509">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NucleicAcidsRes12_6414.pdf">[Full Paper]</a></li>
  <li>Qin B#, Zhou M#, Ge Y#, Taing L, Liu T, Wang Q, Wang S, Chen J, Shen L, Duan X, Hu S, Long H, Zhang Y*, <strong>Liu XS</strong>*. 2012. CistromeMap: A knowledgebase and web server for ChIP-Seq and DNase-Seq studies in mouse and human. Bioinformatics 28(10):1411-2. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/22495751">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Bioinformatics12_1411.pdf">[Full Paper]</a></li>
  <li>Chen Y#, Negre N#, Li Q, Mieczkowska JO, Slattery M, Liu T, Zhang Y, Kim TK, He HH, Zieba J, Ruan Y, Bickel PJ, Myers RM, Wold BJ, White KP*, Lieb JD*, <strong>Liu XS</strong>*. 2012. Systematic evaluation of factors influencing ChIP-seq fidelity. Nat Methods 9(6):609-14. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/22522655">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatMethods12_609.pdf">[Full Paper]</a></li>
  <li>He HH#, Meyer CA#, Chen MW, Jordan VC, Brown M*, <strong>Liu XS</strong>*. 2012. Differential DNase I hypersensitivity reveals factor-dependent chromatin dynamics. Genome Res  22(6):1015-25.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/22508765">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenomeRes12_1015.pdf">[Full Paper]</a></li>
  <li>Feng J#, Liu T#, Qin B, Zhang Y#, <strong>Liu XS</strong>*. 2012. Identifying ChIP-seq enrichment using MACS. Nat Protoc 7(9):1728-40.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/22936215">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatProtoc12_1728.pdf">[Full Paper]</a></li>
  <li>Feng J, Meyer CA, Wang Q, Liu JS, <strong>Liu XS</strong>*, Zhang Y*. 2012. GFOLD: a generalized fold change for ranking differentially expressed genes from RNA-seq data. Bioinformatics 28(12):2782-8.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/22923299">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Bioinformatics12_2782.pdf">[Full Paper]</a></li>
  <li>Meyer CA, Tang Q, <strong>Liu XS</strong>. 2012. Applications of next generation sequencing on studies of nuclear receptor regulation and function. Mol Endocrinol 26(10):1651-9.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/22930692">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/MolEndocrinol12_1651.pdf">[Full Paper]</a></li>
  <li>Bailey ST#, Shin H#, Westerling T, <strong>Liu XS</strong>*, Brown M*. 2012. Estrogen receptor prevents p53-dependent apoptosis in breast cancer. Proc Natl Acad Sci USA109(44):18060-5.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/23077249">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PNAS12_18060.pdf">[Full Paper]</a></li>
  <li>Xu K#, Wu ZJ#, Groner AC, He HH, Cai C, Lis RT, Wu X, Stack EC, Loda M, Liu T, Xu H, Cato L, Thornton JE, Gregory RI, Morrissey M, Vessella RL, Montironi R, Magi-Galluzzi C, Kantoff PW, Balk SP, <strong>Liu XS</strong>*, Brown M*. 2012. EZH2 oncogenic activity in castration-resistant prostate cancer is polycomb-independent. Science 338(6113):1465-9. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/23239736">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Science12_1465.pdf">[Full Paper]</a></li>
  <li>Chen S, Ma J, Wu F, Xiong LJ, Ma H, Xu W, Lu R, Li X, Villen J, Gygi SP, <strong>Liu XS</strong>, Shi Y. 2012. The histone H3 Lys 27 demethylase JMJD3 regulates gene expression by impacting transcriptional elongation. Genes Dev 26(12):1364-75. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/22713873">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenesDev12_1364.pdf">[Full Paper]</a></li>
  <li>Landt SG, Marinov GK, Kundale A, Kheradpour P, Pauli F, Batzoglou S, Bernstein BE, Bickel P, Brown JB, Cayting P, Chen Y, Desalvo G, Epstein C, Fisher-Aylor KI, Euskirchen G, Gerstein M, Gertz J, Hartemink AJ, Hoffman MM, Iyer Vr, Jung YL, Karmakar S, Kellis M, Kharchenko PV, Li Q, Liu T, <strong>Liu XS</strong>, Ma L, et al. 2012. ChIP-seq guidelines and practices of the ENCODE and modENCODE consortial. Genome Res 22(9):1813-31. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/22955991">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenomeRes12_1813.pdf">[Full Paper]</a></li>
  <li>Fu K, Tang W, Feng J, <strong>Liu XS</strong>, Zhang Y. 2012. DiNuP: a systematic approach to reveal differential nucleosome positioning. Bioinformatics 28(15):1965-71. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/22668788">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Bioinformatics12_1965.pdf">[Full Paper]</a></li>
  <li>Benson MJ, Aijo T, Chang X, Gagnon J, Pape UJ, Anatharaman V, Aravind L, Pursiheimo JP, Oberdoerffer S, <strong>Liu XS</strong>, Lahesmaa R, Lahdesmaki H, Rao A. 2012. Heterogeneous nuclear ribonucleoprotein L-like (hnRNPLL) and elongation factor, RNA polymerase II, 2 (ELL2) are regulators of mRNA processing in plasma cells. Proc Natl Acad Sci USA 109(40):16252-7. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/22991471">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PNAS12_16252.pdf">[Full Paper]</a></li>
  <li>Maruyama R, Shipitsin M, Choudhury S, Wu ZJ, Protopopov A, Yao J, Lo PK, Bessarabova M, Ishkin A, Nikolsky Y, <strong>Liu XS</strong>, Sukumar S, Polyak K. 2012. Breast cancer special feature: altered antisense-to-sense transcript ratios in breast cancer. Proc Natl Acad Sci USA 109(8):2820-4. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/21098291">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PNAS12_2820.pdf">[Full Paper]</a></li>
  <li>Xu Y, Xu C, Kato A, Tempel W, Abreu JG, Bian C, Hu Y, Hu D, Zhao B, Cerovina T, Diao J, Wu F, He HH, Cui Q, Clark E, Ma C, Barbara A, Veenstra GJ, Xu G, Kaiser UB, <strong>Liu XS</strong>, Sugrue SP, He X, Min J, Kato Y, Shi YG. 2012. Tet3 CXXC domain and dioxygenase activity cooperatively regulate key genes for xenopus eye and neural development. Cell 151(6):1200-13.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/23217707">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Cell12_1200.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2011">2011</h2>
<ul>
  <li>Tang Q, Chen Y, Meyer CA, Geistlinger T, Lupien M, Wang Q, Liu T, Zhang Y, Brown M*, <strong>Liu XS</strong>*. 2011. A comprehensive view of nuclear receptor cancer cistromes. Cancer Res 71(22):6940-7. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/21940749">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CancerRes11_6940.pdf">[Full Paper]</a></li>
  <li>Chen Y, Meyer CA, Liu T, Li W, Liu JS, <strong>Liu XS</strong>. 2011. MM-ChIP enables integrative analysis of cross-platform and between-laboratory ChIP-chip or ChIP-seq data. Genome Biol 12(2):R11. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/21284836">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenomeBiology11_R11.pdf">[Full Paper]</a></li>
  <li>Liu T, Rechtsteiner A, Egelhofer TA, Vielle A, Latorre I, Cheung MS, Ercan S, Ikegami K, Jensen M, Kolasinska-Zwierz P, Rosenbaum H, Shin H, Taing S,Takasaki T, Iniguez AL, Desai A, Dernburg AF, Kimura H, Lieb JD, Ahringer J, Strome S, <strong>Liu XS</strong>. 2011. Broad chromosomal domains of histone modification patterns in C. elegans. Genome Res 21(2):227-36. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/21177964">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenomeRes11_27.pdf">[Full Paper]</a></li>
  <li>Meyer CA*, He HH, Brown M, <strong>Liu XS</strong>*. 2011. BINOCh: Binding inference from nucleosome occupany changes. Bioinformatics 27(13):1867-8.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/21551136">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Bioinformatics11_1867.pdf">[Full Paper]</a></li>
  <li>Wang X*, Laurie JD, Liu T, Wentz J, <strong>Liu XS</strong>*. 2011. Computational dissection of Arabidopsis smRNAome leads to discovery of novel microRNAs and short interfering RNAs associated with transcription start sites. Genomics 97(4):235-43. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/21295131">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Genomics11_235.pdf">[Full Paper]</a></li>
  <li>Wang X*, <strong>Liu XS</strong>*. 2011. Systematic curation of miRBase annotation using integrated small RNA high-throughput sequencing data for C. elegans and Drosophila. Frontiers in Genetics 2(25):1-14. 
<br><a href="https://liulab-dfci.github.io/resources/publications/Frontiers_Genetics_Wang.pdf">[Full Paper]</a></li>
  <li>Ni M#, Chen Y#, Lim E, Wimberly H, Bailey ST, Imai Y, Rimm DL, <strong>Liu XS</strong>*, Brown M*. 2011. Targeting androgen receptor in estrogen receptor-negative breast cancer. Cancer Cell 20(1):119-31. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/21741601">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CancerCell11_119.pdf">[Full Paper]</a></li>
  <li>Liu T#, Ortiz JA#, Taing L, Meyer CA, Lee B, Zhang Y, Shin HG, Wong SS, Ma J, Lei Y, Pape UJ, Poidinger M, Chen Y, Yeung K, Brown M, Turpaz Y*, <strong>Liu XS</strong>*. 2011. Cistrome: an integrative platform for transcriptional regulation studies. Genome Biol 12(8):R83.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/21859476">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenomeBiol11_R83.pdf">[Full Paper]</a></li>
  <li>Cai C, He HH , Chen S, Ilsa Coleman I, Wang H, Fang Z, Chen S, Nelson PS, <strong>Liu XS</strong>, Brown M, Balk SP. 2011. Androgen receptor gene expression in prostate cancer is directly suppressed by the androgen receptor through recruitment of lysine demethylase 1. Cancer Cell 20(4):457-71. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/22014572">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CancerCell11_57.pdf">[Full Paper]</a></li>
  <li>Feng D*, Liu T*, Sun Z, Bugge A, Mullican SE, Alenghai T, <strong>Liu XS</strong>, Lazar MA. 2011. A circadian rhythm orchestrated by histone deacetylase 3 controls hepatic lipid metabolism. Science 331(6022):1315-9. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/21393543">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Science11_1315.pdf">[Full Paper]</a></li>
  <li>Verzi MP, Shin H, Ho LL, <strong>Liu XS</strong>, Shivdasani RA. 2011. Essential and redundant functions of caudal family proteins in activating adult intestinal genes. Mol Cell Biol 31(10):2026-39. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/21402776">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/MolCellBiol11_2026.pdf">[Full Paper]</a></li>
  <li>Maruyama R, Choudhury S, Kowalczyk A, Bessarabova M, Beresford-Smith B, Conway T, Kaspi A, Wu Z, Nkkolskaya T, Merino VF, Lo PK, <strong>Liu XS</strong>, Nikolsky Y, Sukumar S, Haviv I, Polyak K. 2011. Epigenetic regulation of cell type-specific expression patterns in the human mammary epithelium. PLoS Genetics 7(4):31001369. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/21533021">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PLoS_Genetics11_e1001369.pdf">[Full Paper]</a></li>
  <li>Pastor WA#, Pape UJ#, Huang Y, Henderson HR, Lister R, Ko M, McLoughlin EM, Brudno Y, Mahapatra S, Kapranov P, Tahiliani M, Daley GQ, <strong>Liu XS</strong>, Ecker JR, Milos PM, Agarwal S, Rao A. 2011. Genome-wide mapping of 5-hydroxymethylcytosine in embryonic stem cells. Nature 473(7347):394-7.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/21552279">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Nature_11_394.pdf">[Full Paper]</a></li>
  <li>Marotta LL, Almendro V, Shipitsin M, Schemme J, Walker SR, Bloushtain-Qimron N, Kim JJ, Choudhury SA, Maruyama R, Wu Z, Gonen M, Mulvey LA, Bessarabova MO, Huh SJ, Silver SJ, Kim SY, Park SY, Lee HE, Anderson KS, Richardson AL, Nikolskaya T, Nikolsky Y, <strong>Liu XS</strong>, Root DE, Hahn WC, Frank DA, Polyak K. 2011. The JAK2/STAT3 signaling pathway is required for growth of CD44+CD24- stem cell-like breast cancer cells in human tumors. J Clin Invest 121(7):2723-35. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/21633165">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/JClinInvest11_2723.pdf">[Full Paper]</a></li>
  <li>Adams DJ, Berger B, Harismendy O, Huttenhower C, <strong>Liu XS</strong>, Myers CL, Oshlack A, Rinn JL, Walhout AJ. 2011. Genomics in 2011: Challenges and opportunities. Genome Biol 12(12):137. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/22204305">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenomeBiol11_137.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2010">2010</h2>
<ul>
  <li>He HH#, Meyer C#, Shin H, Bailey ST, Wei G, Wang Q, Zhang Y, Xu K, Ni M, Lupien M, Mieczkowski P, Lieb JD, Zhao K, Brown M*, <strong>Liu XS</strong>*. 2010. Nucleosome dynamics defines transcriptional enhancers. Nat Genet 42(4):343-7.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/20208536">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatureGenetics10_343.pdf">[Full Paper]</a></li>
  <li>Ji H*, <strong>Liu XS</strong>*. 2010. Analyzing 'omics data using hierarchical models. Nature Biotechnology 28(4):337-40.
<br><a href="http://www.nature.com/nbt/journal/v28/n4/abs/nbt.1619.html">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatureBiotech10_337.pdf">[Full Paper]</a></li>
  <li>Vastenhouw NL, Zhang Y, Woods IG, Imam F, Regev A, <strong>Liu XS</strong>*, Rinn J*, Schier AF*. 2010. Chromatin signature of embryonic pluripotency is established during genome activation. Nature 464(7290):922-6.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/20336069">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Nature_10_922.pdf">[Full Paper]</a></li>
  <li>Lupien M#, Meyer CA#, Bailey ST, Eeckhoute J, Cook J, Westerling T, Zhang X, Carroll JS, Rhodes DR, <strong>Liu XS</strong>*, Brown M*. 2010. Growth factor stimulation induces a distinct ER{alpha} cistrome underlying breast cancer endocrine resistance. Genes Dev 24(19):2219-27. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/20889718">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Genes%20Dev10%202219.pdf">[Full Paper]</a></li>
  <li>Cao Q, Zhou M, Wang X, Meyer CA, Zhang Y, Chen Z, Li C*, <strong>Liu XS</strong>*. 2010. CaSNP: a database for interrogating copy number alterations of cancer from SNP array data. Nucleic Acids Res 39(Database issue):D968-74. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/20972221">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NucleicAcidsRes11_D968.pdf">[Full Paper]</a></li>
  <li>Wu ZJ#, Meyer CA#, Choudhury S, Shipitsin M, Bessarabova M, Nikolsky Y, Sukumar S, Schwartzman A, Liu JS, Polyak K*, <strong>Liu XS</strong>*. 2010. Gene expression profiling of human breast tissue samples using SAGE-Seq. Genome Res 20(12):1730-9. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/21045080">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/GenomeRes10_1730.pdf">[Full Paper]</a></li>
  <li>The modENCODE Consortium (<strong>Liu XS</strong>*). 2010. Integrative analysis of the Caenorhabditis elegans genome by the modENCODE project. Science 330(6012):1775-87.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/21177976">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Science10_1775.pdf">[Full Paper]</a></li>
  <li>Bienvenu F, Jirawatnotai S, Elias JE, Meyer CA, Mizeracka K, Marson A, Frampton GM, Cole MF, Odom D, Odajima J, Geng Y, Zagozdzon A, Jecrois M, Young RA, <strong>Liu XS</strong>, Cepko C, Gygi SP, Sicinski P. 2010. Transcriptional role of cyclin D1 in development revealed by a genetic-proteomic screen. Nature 463(7279):374-78. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/20090754">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Nature10_374.pdf">[Full Paper]</a></li>
  <li>Hirsch HA, Iliopoulos D, Joshi A, Zhang Y, Jaeger SA, Bulyk M, <strong>Liu XS</strong>, Struhl K. 2010. A transcriptional signature and common gene networks link cancer with lipid metabolism and diverse human diseases. Cancer Cell 17(4):348-61. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/20385360">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CancerCell10_348.pdf">[Full Paper]</a></li>
  <li>Wolfer A, Wittner B, Irimia D, Flavin RJ, Lupien M, Gunawardane RN, Meyer CA, Lightcap ES, Tamayo P, Mesirov JP, <strong>Liu XS</strong>, Shioda T, Toner M, Loda M, Brown M, Bruggs JS, Ramaswamy S. 2010. MYC regulation of a "poor-prognosis" metastatic cancer cell state. Proc Natl Acad Sci U S A 107(8):3698-703. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/20133671">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PNAS10_3698.pdf">[Full Paper]</a></li>
  <li>Ahmadiyeh N, Pomerantz MM, Grisanzio C, Herman P, Jia L, Almendro V, He HH, Brown M, <strong>Liu XS</strong>, Davis M, Caswell JL, Beckwith CA, Hills A, Macconaill L, Coetzee GA, Regan MM, Freedman ML. 2010. 8q24 prostate, breast and colon risk loci show tissue-specific long-range interaction with MYC. Proc Natl Acad Sci U S A 107(21):9742-6. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/20453196">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/2010_Ahmadiyeh_PNAS_9742.pdf">[Full Paper]</a></li>
  <li>Verzi MP, Hatzis P, Sulahian R, Philips J, Schuijers J, Shin H, Freed E, Brown MA, Lynch JP, Dang DT, Clevers H, <strong>Liu XS</strong>, Shivdasani RA. 2010. TCF4 and CDX2, major transcription factors for intestinal function, converge on the same cis-regulatory regions. Proc Natl Acad Sci U S A 107(34):15157-62. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/20696899">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PNAS107_34.pdf">[Full Paper]</a></li>
  <li>Fan X, Moqtaderi Z, Jin Y, Zhang Y, <strong>Liu XS</strong>, Struhl K. 2010. Nucleosome depletion at yeast terminators is not intrinsic and can occur by a transcriptional mechanism linked to 3'-end formation. Proc Natl Acad Sci USA 107(42):17945-50. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/20921369">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PNAS10_17945.pdf">[Full Paper]</a></li>
  <li>Verzi MP,#, Shin HG#, He HH, Meyer CA, Montgomery RK, Fleet JC, Brown MA, <strong>Liu XS</strong>, Shivdasani RA. 2010. Differentiation-specific histone modifications reveal dynamic chromatin interactions and partners for the intestinal transcription factor CDX2. Dev Cell 19(5):713-26.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/21074721">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Dev%20Cell10_713.pdf">[Full Paper]</a></li>
  <li>Ko M, Huang Y, Jankowska A, Pape UJ, Tahiliani M, Bandukwala HS, An J, Lamperti ED, Koh KP, <strong>Liu XS</strong>, Aravind L, Agarwal S, Maciejewki JP, Rao A. 2010. Impaired hydroxylation of 5-methylcytosine in myeloid neoplasms with mutant TET2. Nature 468(7325):839-43.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/21057493">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Nature10_839.pdf">[Full Paper]</a></li>
  <li>Zhang Y, Moqtaderi Z, Rattner B, Euskirchen G, Snyder M, Kadonaga JT, <strong>Liu XS</strong>, Struhl K. 2010. Evidence against a genomic code for nucleosome positioning. Reply to: "Nucleosome sequence preferences influence in vivo nucleosome organization". Nat Struct Mol Biol 17:920-23).
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/20683474">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/NatStrucMolBiol17_8.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2009">2009</h2>
<ul>
  <li>Johnson WE, Liu JS, <strong>Liu XS</strong>. 2009. Doubly-stochastic continuous-time hidden Markov approach for applications on genome tiling arrays. Annals of Applied Statistics 3(3):1183-1203.
<br><a href="https://liulab-dfci.github.io/resources/publications/2009_Johnson_AnnApplStat_1183.pdf">[Full Paper]</a></li>
  <li><strong>Liu XS</strong>, Meyer CA. 2009. ChIP-Chip: algorithms for calling binding sites. Methods Mol Biol 556-75. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/19488878">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/MethodsMolBiol09_165.pdf">[Full Paper]</a></li>
  <li>Shin H, Liu T, Manrai AK, <strong>Liu XS</strong>. 2009. CEAS: Cis-regulatory element annotation system. Bioinformatics 25(19):2605-6.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/19689956">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/2009_Shin_CEAS_Bio.pdf">[Full Paper]</a></li>
  <li>Wang Q, Li W, Zhang Y, Yuan X, et al., Kantoff PW, <strong>Liu XS</strong>*, Brown M*. 2009. Androgen receptor regulates a distinct transcription program in androgen-independent prostate cancer. Cell 138(2):245-56. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/19632176">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/2009_Wang_ReprogAndRecpt_C.pdf">[Full Paper]</a></li>
  <li>Boros J, Donaldson IJ, O'Donnel A, Zeef L, Lupien M, Meyer CA, <strong>Liu XS</strong>, Brown M, Sharrocks AD. 2009. Elucidation of the Elk-1 target gene network reveals a role in the coordinate regulation of core components of the gene regulation machinery. Genome Res 19(11):1963-73. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/19687146">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Genome%20Res09_1963.pdf">[Full Paper]</a></li>
  <li>Eeckhoute J, Lupein M, Meyer CA, Verzi MP, Shivdasani RA, <strong>Liu XS</strong>, Brown M. 2009. Cell-type selective chromatin remodeling defines the active subset of FoxA1-bound enhancers. Genome Res 19(3)372-80. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/19129543">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/2009_Eeckhoute_RemodFoxA1_GR.pdf">[Full Paper]</a></li>
  <li>Huang J, Chen T, Liu X, Yuan D, Jiang J, Li J, Li D, <strong>Liu XS</strong>, Li W, Kang J, Pei G. 2009. More synergetic cooperation of Yamanaka factors in induced pluripotent stem cells than in embryonic stem cells. Cell Res 19(10):1127-38.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/19736564">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/CellRes09_1127.pdf">[Full Paper]</a></li>
  <li>Kolasinska-Zwierz P, Down T, Latorre I, Liu T, <strong>Liu XS</strong>, Ahringer J. 2009, Differential chromatin marking of introns and expressed exons by H3K36me3. Nat Genet 41(3):376-81. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/19182803">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Kolasinska_H3K36me3_NatGen_2009.pdf">[Full Paper]</a></li>
  <li>Lupien M, Eeckhoute J, Meyer CA, Krum SA, Rhodes, DR, <strong>Liu XS</strong>, Brown M. 2009. Coactivator function defines the active estrogen receptor alpha cistrome. Moll Cell Biol 29(12):3413-23. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/19364822">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/2009_Lupien_CoactivFunct_MCB.pdf">[Full Paper]</a></li>
  <li>Wang X, Elling A, Li X, Li N, Peng Z, He G, Sun H, Qi Y, <strong>Liu XS</strong>, Deng XW. 2009. Genome-wide and organ-specific landscapes of epigenetic modifications and their relationships to mRNA and small RNA transcriptomes in maize. Plant Cell 21(4):1053-69.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/19376930">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/2009_Wang_Maize_PC.pdf">[Full Paper]</a></li>
  <li>Xia X, Lemieux M, Li W, Carroll JS, Brown M, <strong>Liu X</strong>, Kung AL. 2009. Integrative analysis of HIF binding and transactivation reveals its role in maintaining histone methylation homeostatis. Proc Natl Acad Sci USA 106(11):4260-5. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/19255431">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Xia_HIF_Binding_PNAS_2009.pdf">[Full Paper]</a></li>
  <li>Zhang Y, Moqtaderi Z, Rattner B, Zhenhai Zhang Z, Euskirchen G, Snyder M, Kadonaga JT, Pugh BF, <strong>Liu XS</strong>, Struhl K. 2009. Intrinsic histone-DNA interactions are not the major determinant of nucleosome positions in vivo: evidence against a nucleosome code. Nat Struct Mol Biol 16(8):847-52. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/19620965">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/2009_Zhang_IntrinHisDNA_NSMB.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2008">2008</h2>
<ul>
  <li>Johnson DS, Li W, et al., Struhl K, Myers RM, Lieb JD, <strong>Liu XS</strong>. 2008. Systematic evaluation of variability in ChIP-chip experiments using predefined DNA targets. Genome Res 18(3):393-403. 
<br><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=18258921">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Johnson_Spikein_GR_2008.pdf">[Full Paper]</a></li>
  <li>Gottardo R, Li W, Johnson WE, <strong>Liu XS</strong>. 2008. A flexible and powerful Bayesian hierarchical model for ChIP-chip experiments. Biometrics 64(2):468-78.
<br><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=17888037">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Gottardo_Flexible_Biomet_2008.pdf">[Full Paper]</a></li>
  <li>Li W, Carroll JS, Brown M, <strong>Liu XS</strong>. 2008. xMAN: extreme Mapping of OligoNucleotides. BIOCOMP '07 (acceptance rate 27%). BMC Genomics 9 Suppl 1:S20. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/18366610?dopt=Citation">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Li_XMAN_BMC_2008.pdf">[Full Paper]</a></li>
  <li>Zhang Y*, Liu T*, Meyer CA, Eeckhoute J, Johnson DS, Bernstein B, Nusbaum C, Myers RM, Brown M, Li W, <strong>Liu XS</strong>. 2008. Model-based Analysis of ChIP-seq (MACS).Genome Biol 9(9):R137. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/18798982">[Abstract]</a>
<a href="http://genomebiology.com/2008/9/9/R137">[Full Paper]</a></li>
  <li>Zhang Y*, Shin H*, Song JS, Lei Y, <strong>Liu XS</strong>. 2008. Identifying positioned nucleosomes with epigenetic marks in human from ChIP-Seq. BMC Genomics 9:537. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/19014516">[Abstract]</a>
<a href="http://www.biomedcentral.com/1471-2164/9/537">[Full Paper]</a></li>
  <li>Agoulnik IU, Bingman WE, Nakka M, Li W, Wang Q, <strong>Liu XS</strong>, Brown M, Weigel NL. 2008. Target gene specific regulation of androgen receptor activity by p42/p44 MAPK. Mol Endocrinol 22(11):2420-32. 
<br><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=18787043">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/MolEndocrinol08_2420.pdf">[Full Paper]</a></li>
  <li>Holmes KA, Song JS, <strong>Liu XS</strong>, Brown M, Carroll JS. 2008. Nkx3-1 and LEF-1 function as transcriptional inhibitors of Estrogen Receptor Activity. Cancer Res 68(18):7380-5.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/18794125?ordinalpos=1&amp;itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Holmes_Nkx3-1_CanRes_2008.pdf">[Full Paper]</a></li>
  <li>Lefterova M, Zhang Y, Steger DJ, Schupp M, Schug J, Cristancho A, Feng D, Zhuo D, Stoeckert CJ, <strong>Liu XS</strong>, Lazar MA. 2008. PPARgamma and C/EBP factors orchestrate adipocyte biology via adjacent binding on a genome-wide scale. Genes Dev 22(21):2941-2952. 
<br><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=18981473">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Lefterova_PPARgamma_GenDev_2008.pdf">[Full Paper]</a></li>
  <li>Lupien M, Eeckhoute J, Meyer CA, Wang Q, Zhang Y, Li W, Carroll JS, <strong>Liu XS</strong>, Brown M. 2008. FoxA1 translates epigenetic signatures into enhancer driven lineage-specific transcription. Cell 132(6):958-70. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/18358809?dopt=Citation">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Lupien_FoxA1_Cell_2008.pdf">[Full Paper]</a></li>
  <li>Ozsolak F, Poling L, Wang Z, Liu H, <strong>Liu XS</strong>, Roeder RG, Zhang X, Song JS, Fisher DE. 2008. Chromatin structure analyses identify miRNA promoters. Genes Dev 22(22):3172-83. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/19056895">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Ozsolak_microRNA_GenDev_2008.pdf">[Full Paper]</a></li>
  <li>Song J, Liu X, <strong>Liu XS</strong>, He X. 2008. A high-resolution map of nucleosome positioning on a fission yeast centromere. Genome Res 18(7):1064-72. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/18411404">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Song_PombeCentromereNucleosome_GR_2008.pdf">[Full Paper]</a></li>
  <li>Stayrook KR, Savkur RS, Su C, Varga G, Bu X, Wei T, Nagpal S, <strong>Liu XS</strong>, Burris T. 2008. Regulation of human 3a-hydroxysteroid dehydrogenase (AKR1C4) gene expression by the liver X receptor a. Molecular Pharmacology 73(2):607-12. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/18024509">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Molecular%20Pharmacol08_607.pdf">[Full Paper]</a></li>
  <li>Yagi S, Hirabayashi K, Sato S, Li W, Takahashi Y, Hirakawa T., Hattori N, Ohgane J, Tanaka S, <strong>Liu XS</strong>, Shiota K. 2008. DNA methaylation profile of tissue-dependent and differentially methylated regions (T-DMRs) in mouse promoter regions demonstrating tissue-specific gene expression. Genome Res 18(12):1969-78. 
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/18971312">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Genome%20Res08_1969.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2007">2007</h2>
<ul>
  <li><strong>Liu XS</strong>. 2007. Getting started in tiling microarray analysis. PLoS Comput Biol 3(10):1842-4.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/17967045">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PLoS%20Comput%20Biol07_1842.pdf">[Full Paper]</a></li>
  <li>Song JS, Johnson WE, Zhu X, Zhang X, Li W, Manrai AK, Liu JS, Chen R, <strong>Liu XS</strong>. 2007. Model based analysis for 2-color arrays (MA2C). Genome Biol 8(8):R178.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/17727723">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Song_MA2C_GenomeBiol_2007.pdf">[Full Paper]</a></li>
  <li>Song J, Maghsoudi K, Li W, Fox A, Quackenbush J and <strong>Liu XS</strong>. 2007. Microarray blob-defect removal improves array analysis. Bioinformatics 23(8):966-71.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/17332024">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Song_Blob_Bioinformatics.pdf">[Full Paper]</a></li>
  <li>Ozsolak F*, Song JS*, <strong>Liu XS</strong>#, Fisher DE#. 2007. High-throughput mapping of the chromatin structure of human promoters. Nat Biotech 25(2)244-8.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/17220878">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Ozsolak_Promoter_NBT_2007.pdf">[Full Paper]</a></li>
  <li>He H, Wang J, Liu T, <strong>Liu X</strong>, Li T, Wang Y, Qian Z, Zheng H, Zhu X, Wu T, Shi B, Deng W, Zhou W, Skogerbo G, Chen R. 2007. Mapping the C. elegans noncoding transcriptome with a whole genome tiling microarray. Genome Res 17(10):1471-7.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/17785534">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/He_elegansNcRNA_GR_2007.pdf">[Full Paper]</a></li>
  <li>Litovchick L, Sadasivam S, Florens L, Zhu X, Swanson SK, Velmurugan S, Chen R, Washburn MP, <strong>Liu XS</strong>, DeCaprio JA. 2007. Evolutionally conserved multi-subunit RBL2/p130 and E2F4 protein complex represses human cell cycle-dependent genes in quiescence. Mol Cell 26(4)539-51.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/17531812">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Litovchick_E2F4_MolCell.pdf">[Full Paper]</a></li>
  <li>O'Neil J, Tchinda J, Gutierrez A, Moreau L, Maser R, Wong KK, Li W, KcKenna K, <strong>Liu X</strong>, Neuberg D, Silverman L, DeAngelo D, Kutok J, Rothstein R, DePinho R, Chin L, Lee C, Look AT. 2007. Alu elements mediate MYB gene tandem duplication in human T-ALL. J Exp Med 204(13):3059-66.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/18070937">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/ONeil_MybDupTALL_JEM_2007.pdf">[Full Paper]</a></li>
  <li>Wang Q, Li W, <strong>Liu XS</strong>, Carroll JS, Janne OA, Keeton EK, Chinnaiyan AM, Pienta KJ and Brown M. 2007. A hierarchical network of transcription factors governs androgen receptor dependent prostate cancer growth. Mol Cell 27(3):380-92.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/17679089">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Wang_ARChr2122_MolCel_2007.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2006">2006</h2>
<ul>
  <li>Carroll JS, Meyer CA, Song J, Li W, Geistlinger TR, Eeckhoute J, Brodsky AS,KeetonEK, Fertuck KC,Hall GF, Wang Q, Bekiranov S, Sementchenko V, Fox EA, Silver PA, Gingeras TR, <strong>Liu XS</strong>*, Brown M*. 2006. Genome-wide analysis of estrogen receptor binding sites. Nat Genet 38(11):1289-97.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/17013392">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Carroll_ER_NG.pdf">[Full Paper]</a></li>
  <li>Ji X*, Li W*, Song J, Wei L, <strong>Liu XS</strong>. 2006. CEAS: cis-regulatory element annotation system. Nucleic Acids Res 34(Web Server Issue):W551-4.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845068">[Abstract]</a>
<a href="http://nar.oxfordjournals.org/cgi/reprint/34/suppl_2/W551.pdf">[Full Paper]</a></li>
  <li>Johnson WE*, Li W*, Meyer CA*, Gottardo R, Carroll JS, Brown M, <strong>Liu XS</strong>. 2006. MAT: Model-based analysis of tiling-arrays for ChIP-chip. Proc Natl Acad Sci USA 103(33)12457-62.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/16895995">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Johnson_MAT_PNAS.pdf">[Full Paper]</a></li>
  <li>Lieb JD, Beck S, Bulyk ML, Farnham P, Hattori N, Henikoff S, <strong>Liu XS</strong>, Okumura K, Shiota K, Ushijima T, Greally JM. 2006. Applying whole-genome studies of epigenetic regulation to study human disease. Cytogenet. Genome Res 114(1):1-15.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/16717444">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Lieb_GenomeEpi_Cyto_2006.pdf">[Full Paper]</a></li>
  <li>Nelson EA, Walker SR, Li W, <strong>Liu XS</strong>, Frank DA. 2006. Identification of human STAT5-dependent gene regulatory elements based on interspecies homology. J Biol Chem  281(36):26216-24.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/16840779">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Nelson_STAT5_JBC.pdf">[Full Paper]</a></li>
  <li>Palomero T, Odom DT, O'Neil J, Ferrando AA, Margolin A, Neuberg DS, Winter SS, Larson RS,Li W, <strong>Liu XS</strong>, Young RA, Look AT. 2006. Transcriptional regulatory networks downstream of TAL1/SCL in T-cell acute lymphoblastic leukemia. Blood 108(3):986-92.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/16621969">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Palomero_TAL1_Blood.pdf">[Full Paper]</a></li>
  <li>Swinburn IA, Meyer CA, <strong>Liu XS</strong>, Silver PA, Brodsky AS. 2006. Genomic localization of RNA-binding proteins reveals links between pre-mRNA processing and transcription. Genome Res 16(7):912-21.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/16769980">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Swinbrune_RNABinding_GenomeRes.pdf">[Full Paper]</a></li>
  <li>Zhang Y, <strong>Liu XS</strong>, Liu Q, Wei L. 2006. Genome-wide in silico identification and analysis of cis Natural Antisense Transcripts (cis-NATs) in Ten Species. Nucleic Acids Res 34(12):3465-75.<br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/16849434">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Zhang_Antisense_NAR.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2005">2005</h2>
<ul>
  <li>Brodsky AS, Meyer CA, Hall G, Swinburne IA, Keenan BJ, <strong>Liu XS</strong>. 2005. Fox EA, Silver PA. Genomic mapping of RNA polymerase II reveals sites of co-transcriptional regulation in human cells. Genome Biol 6(8):R64.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/16086846">[Abstract]</a></li>
  <li><strong>Liu XS</strong>. 2005. Gibbs sampling and bioinformatics. Encyclopedia of Genetics, Genomics, Proteomics and Bioinformatics. John Wiley &amp; Sons Ltd, UK ISBN: 0-470-84974-6.
<br><a href="http://www.wiley.com/WileyCDA/WileyTitle/productCd-0470849746.html">[Full Paper]</a></li>
  <li>Li W, Meyer CA, <strong>Liu XS</strong>. 2005. A hidden Markov model for analyzing ChIP-chip experiments on genome tiling arrays and its application to p53 binding sequences. Bioinformatics 21 Suppl.1:i274-i282.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/15961467">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Li_p53TilingHMM_ISMB_2005.pdf">[Full Paper]</a></li>
  <li>Ben-Yehuda S, Jujita M, <strong>Liu XS</strong>, Gorbatyuk B, Skoko D, Yan J, Eichenberger P, Rudner DZ, Marko JF, Liu JS, Losick R. 2005. Defining a centromere-like element in Bacillus subtilis by identifying the binding sites for the chromosome-anchoring protein RacA. Mol Cell 17(6):773-82.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/15780934">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/BenYehuda_RacA_MolCel_2005.pdf">[Full Paper]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Brodsky_PolII_GenBio_2005.pdf">[Full Paper]</a></li>
  <li>Carroll JS, <strong>Liu XS</strong>, Brodsky AS, Li W, Meyer CA, Szary AJ, Shao W, Hestermann EV, Geistlinger TR, Fox EA, Silver PA, Brown M. 2005. Chromosome-wide mapping of estrogen receptor binding reveals long-range regulation requiring the forkhead protein FoxA1. Cell 122(1):33-43.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/16009131">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Carroll_ERTiling_Cell_2005.pdf">[Full Paper]</a></li>
  <li>Hong P, <strong>Liu XS</strong>, Zhou Q, Lu X, Liu JS, Wong WH. 2005. A boosting approach for motif modeling using ChIP-chip data. Bioinformatics 21(11):2636-43.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/15817698">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Hong_MotifBooster_Bioinfo_2005.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2004">2004</h2>
<ul>
  <li>Liu Y*, <strong>Liu XS</strong>*, Wei L, Altman RB, Batzoglou S. 2004. Eukaryotic regulatory element conservation analysis and identification using comparative genomics. Genome Res 14(3):451-8.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/14993210">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Liu_CP_GR_2004.pdf">[Full Paper]</a></li>
  <li>Liu Y, Wei L, Batzoglou S, Brutlag DL, Liu JS, <strong>Liu XS</strong>. 2004. A suite of web-based programs to search for transcriptional regulatory motifs. Nucleic Acids Res 32(Web Server Issue):W204-7.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/15215381">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Liu_WebMotifFinders_NAR_2004.pdf">[Full Paper]</a></li>
  <li>Jensen ST, <strong>Liu XS</strong>, Zhou Q, Liu JS. 2004. Computational discovery of gene regulatory binding motifs: a Bayesian perspective. Stat Sci 19(1):188-204.
<br><a href="https://liulab-dfci.github.io/resources/publications/Jensen_Moitf_StatSci_2004.pdf">[Full Paper]</a></li>
  <li>Yildiz FH, <strong>Liu XS</strong> , Heydorn A, Schoolnik GK. 2004. Molecular Analysis of Rugosity in a Vibrio cholerae O1 El Tor phase variant. Mol Microbiol 53(2):497-515.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/15228530">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Yildiz_VCPhase_MM_2004.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2003">2003</h2>
<ul>
  <li>Conlon EM*, <strong>Liu XS</strong>*, Lieb JD, Liu JS. 2003. Integrating regulatory motif discovery and genome-wide expression analysis. Proc Nat Acad Sci USA 100(6):3339-44.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/12626739">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Conlon_MotifRegressor_PNAS_03.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2002">2002</h2>
<ul>
  <li><strong>Liu XS</strong>*, Brutlag DL, Liu JS. 2002. An algorithm for finding protein-DNA binding sites with applications to chromatin immunoprecipitation microarray experiments. Nat Biotech 20(8):835-9. *Liu X added middle name Shirley.<br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/12101404">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Liu_MDscan_NatBiotech_2002.pdf">[Full Paper]</a></li>
  <li>Liu JS, Gupta M, <strong>Liu X</strong>, Mayerhofere L, Lawrence CE. 2002. Statistical models for biological sequence motif discovery. Case Studies in Bayesian Statistics, IV, 2002, Springer-Verlag, New York.
<br><a href="https://liulab-dfci.github.io/resources/publications/Liu_Motif_Case_2003.pdf">[Full Paper]</a></li>
</ul>

<h2 id="2001">2001</h2>
<ul>
  <li>Lieb JD, <strong>Liu X</strong>, Botstein D, Brown PO. 2001. Promoter-specific binding of Rap1 revealed by genome-wide maps of protein-DNA association. Nat Genet 28(4):327-34.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/11455386">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Lieb_Rap1_NatGen_2001.pdf">[Full Paper]</a></li>
  <li><strong>Liu X</strong>, Brutlag DL, Liu JS. 2001. BioProspector: discovering conserved DNA motifs in upstream regulatory regions of co-expressed genes. Proc Pac Symp Biocomput 127-38.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/11262934">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Liu_BioProspector_PSB_2001.pdf">[Full Paper]</a></li>
</ul>

<h2 id="1999">1999</h2>
<ul>
  <li>Dugan JM, Berrios DC, <strong>Liu X</strong>, Kim DK, Kaizer H, Fagan LM. 1999. Automation and integration of components for generalized semantic markup of electronic medical texts. Proc AMIA Symp 736-40.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/10566457">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Dugan_Markup_AMIA_1999.pdf">[Full Paper]</a></li>
  <li>Fischer P, <strong>Liu X</strong>, Lizotte-Waniewski M, Kamal IH, Ramzy RM, Williams SA. 1999. Development of a quantitative, competitive polymerase chain reaction-enzyme-linked immunosorbent assay for the detection of Wuchereria bancrofti DNA. XParasitol Res 85(3):176-83.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/9951959">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/Fischer_PCRELISA_ParasRes_1999.pdf">[Full Paper]</a></li>
</ul>

<h2 id="1998">1998</h2>
<ul>
  <li><strong>Liu X</strong>, Altman RB. 1998. Updating a bibliography using the related articles function within PubMed. Proc AMIA Symp 750-4.
<br><a href="http://www.ncbi.nlm.nih.gov/pubmed/9929319">[Abstract]</a>
<a href="https://liulab-dfci.github.io/resources/publications/PNAS1998_750.pdf">[Full Paper]</a></li>
</ul>

  </div>
</div>



    <footer>
  <div class="container beautiful-jekyll-footer">
    <div class="row">
      <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
        <ul class="list-inline text-center footer-links"><li><a href="https://liulab-dfci.github.io/feed.xml" title="RSS"><span class="fa-stack fa-lg" aria-hidden="true">
                  <i class="fa fa-circle fa-stack-2x"></i>
                  <i class="fa fa-rss fa-stack-1x fa-inverse"></i>
                </span>
                <span class="sr-only">RSS</span>
              </a>
            </li><li><a href="https://github.com/liulab-dfci" title="GitHub"><span class="fa-stack fa-lg" aria-hidden="true">
                  <i class="fa fa-circle fa-stack-2x"></i>
                  <i class="fa fa-github fa-stack-1x fa-inverse"></i>
                </span>
                <span class="sr-only">GitHub</span>
              </a>
            </li><li><a href="https://twitter.com/xshirleyliu" title="Twitter"><span class="fa-stack fa-lg" aria-hidden="true">
                  <i class="fa fa-circle fa-stack-2x"></i>
                  <i class="fa fa-twitter fa-stack-1x fa-inverse"></i>
                </span>
                <span class="sr-only">Twitter</span>
              </a>
            </li></ul>
      <p class="copyright text-muted">
      Xiaole Shirley Liu Lab
      &nbsp;•&nbsp;
      2023

      

      
      </p>
          <!-- Please don't remove this, keep my open source work credited :) -->
    <p class="theme-by text-muted">
      Theme by
      <a href="https://deanattali.com/beautiful-jekyll/">beautiful-jekyll</a>
    </p>
      </div>
    </div>
  </div>
</footer>

  
    


  
    <!-- doing something a bit funky here because I want to be careful not to include JQuery twice! -->
    
      <script>
      	if (typeof jQuery == 'undefined') {
          document.write('<script src="/js/jquery-1.11.2.min.js"></scr' + 'ipt>');
      	}
      </script><script src="./Publications_files/jquery-1.11.2.min.js"></script>
    
  
    <!-- doing something a bit funky here because I want to be careful not to include JQuery twice! -->
    
      <script src="./Publications_files/bootstrap.min.js"></script>
    
  
    <!-- doing something a bit funky here because I want to be careful not to include JQuery twice! -->
    
      <script src="./Publications_files/main.js"></script>
    
  






  
  

</body></html>