<!DOCTYPE html>
<!-- saved from url=(0039)https://liulab-dfci.github.io/software/ -->
<html lang="en"><!-- Beautiful Jekyll | MIT license | Copyright Dean Attali 2016 --><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, viewport-fit=cover">

  <title>Software</title>

  <meta name="author" content="Xiaole Shirley Liu Lab">

  
  <meta name="description" content="Softwares in Liu Lab">
  

  <link rel="alternate" type="application/rss+xml" title="Liu Lab - Xiaole Shirley Liu Lab at DFCI" href="https://liulab-dfci.github.io/feed.xml">

  

  

  


  
    
      
  <link rel="stylesheet" href="./Software_files/font-awesome.min.css">


    
  

  
    
      <link rel="stylesheet" href="./Software_files/bootstrap.min.css">
    
      <link rel="stylesheet" href="./Software_files/bootstrap-social.css">
    
      <link rel="stylesheet" href="./Software_files/main.css">
    
  

  
    
      <link rel="stylesheet" href="./Software_files/css">
    
      <link rel="stylesheet" href="./Software_files/css(1)">
    
  

  

  

  

    <!-- Facebook OpenGraph tags -->
  

  
  <meta property="og:title" content="Software">
  

   
  <meta property="og:description" content="Softwares in Liu Lab">
  


  <meta property="og:type" content="website">

  
  <meta property="og:url" content="https://liulab-dfci.github.io/software/">
  <link rel="canonical" href="https://liulab-dfci.github.io/software/">
  

  


  <!-- Twitter summary cards -->
  <meta name="twitter:card" content="summary">
  <meta name="twitter:site" content="@">
  <meta name="twitter:creator" content="@">

  
  <meta name="twitter:title" content="Software">
  

  
  <meta name="twitter:description" content="Softwares in Liu Lab">
  

  

  

  
  <link rel="icon" href="https://liulab-dfci.github.io/img/favicon.ico" type="image/x-icon">
  <link rel="shortcut icon" href="https://liulab-dfci.github.io/img/favicon.ico" type="image/x-icon">
</head>


  <body>

    

  
    <nav class="navbar navbar-default navbar-fixed-top navbar-custom">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#main-navbar">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button><a class="navbar-brand" href="https://liulab-dfci.github.io/">Liu Lab</a></div>

    <div class="collapse navbar-collapse" id="main-navbar">
      <ul class="nav navbar-nav navbar-right">
          <li><a href="https://liulab-dfci.github.io/">Home</a></li>
          <li><a href="https://liulab-dfci.github.io/software">Software</a></li>
          <li><a href="https://liulab-dfci.github.io/news">News &amp; Events</a></li>
          <li><a href="https://liulab-dfci.github.io/people">People</a></li>
          <li><a href="https://liulab-dfci.github.io/publication">Publications</a></li>
          <li><a href="https://liulab-dfci.github.io/teaching">Teaching</a></li>
          <li><a href="https://liulab-dfci.github.io/careers">Careers</a></li>
          <li><a href="https://liulab-dfci.github.io/contact">Contact</a></li></ul>
    </div>

	

  </div>
</nav>


        <div class="intro-header">
  <div role="main" class="container">
    <h1 id="software">Software</h1>

<h3 id="repositories">Repositories</h3>
<ul>
  <li><a href="https://github.com/liulab-dfci?tab=repositories"><strong>GitHub</strong></a> &nbsp;</li>
  <li><a href="https://bitbucket.org/liulab/workspace/repositories"><strong>BitBucket</strong></a> &nbsp;</li>
</ul>

<h3 id="single-cell">Single Cell</h3>
<ul>
  <li><a href="http://github.com/liulab-dfci/MAESTRO"><strong>MAESTRO</strong></a> &nbsp;
Model-based Analyses of Transcriptome and Regulome (MAESTRO) is a comprehensive open-source computational workflow for integrative analysis of single-cell RNA-seq (scRNA-seq) and ATAC-seq (scATAC-seq) data from multiple platforms.</li>
  <li><a href="https://github.com/yi-zhang/MetaTiME"><strong>MetaTiME</strong></a> &nbsp;
Meta-components in Tumor immune MicroEnvironment: MetaTiME learns data-driven, interpretable, and reproducible gene programs by integrating millions of single cells from hundreds of tumor scRNA-seq data.</li>
  <li><a href="https://github.com/cistrome/MIRA"><strong>MIRA</strong></a> &nbsp;
Multimodal models for Integrated Regulatory Analysis, or MIRA, is a python package for analyzing the dynamic processes of gene regulation using single-cell multiomics datasets.</li>
  <li><a href="https://huggingface.co/ctheodoris/Geneformer"><strong>Geneformer</strong></a> &nbsp;
foundational transformer model pretrained on a large-scale corpus of single cell transcriptomes to enable context-aware predictions in settings with limited data in network biology.</li>
</ul>

<h3 id="cancer-immunotherapy">Cancer Immunotherapy</h3>
<ul>
  <li><a href="https://cistrome.shinyapps.io/timer/"><strong>TIMER</strong></a> &nbsp; and <a href="http://timer.cistrome.org/"><strong>TIMER2</strong></a> &nbsp; <a href="http://timer.comp-genomics.org/">[<strong>TIMER2 (Korea)</strong>]</a> &nbsp; <a href="http://timer2.compbio.cn/">[<strong>TIMER2 (China)</strong>]</a> &nbsp; Tumor IMmune Estimation Resource (TIMER) is a comprehensive resource for the clinical relevance of tumor-immune infiltrates, including B cells, CD4 and CD8 T cells, neutrophils, macrophages and dendritic cells.</li>
  <li><a href="https://bitbucket.org/liulab/trust/"><strong>TRUST</strong></a> &nbsp; and latest version <a href="https://github.com/liulab-dfci/TRUST4"><strong>TRUST4</strong></a> &nbsp; (<a href="https://drive.google.com/file/d/1g_CKX8zctjWvRbEJa1Ec_DCrEzfzpJxN/view?usp=sharing">tutorial video</a>) 
is a computational algorithm that can extract TCR/BCR immune repertoires from tumor bulk or single cell RNA-seq data.</li>
  <li><a href="https://bitbucket.org/liulab/deepbcr/src/master/"><strong>DeepBCR</strong></a> &nbsp; Deep learning framework for cancer-type classification and binding affinity estimation using B cell receptor repertoires</li>
  <li><a href="http://tide.dfci.harvard.edu/"><strong>TIDE</strong></a> &nbsp;
TIDE is a gene expression biomarker to predict the clinical response to immune checkpoint blockade. TIDE also provides gene-wise associations with T-cell dysfunction and immunotherapy resistance, computed from a large amount of cancer genomics datasets.</li>
  <li><a href="http://tismo.cistrome.org/"><strong>TISMO</strong></a> &nbsp;
Tumor Immune Syngeneic MOuse (TISMO) is a database specifically designed for hosting, visualizing, and analyzing an extensive collection of syngeneic mouse model data.</li>
  <li><a href="https://github.com/liulab-dfci/RIMA_pipeline"><strong>RIMA</strong></a> &nbsp;
RNA-seq IMmune Analysis (RIMA) pipeline integrative computational analysis of bulk RNA-sequencing data to characterize tumor immunity.</li>
</ul>

<h3 id="cancer-precision-medicine">Cancer Precision Medicine</h3>
<ul>
  <li><a href="http://care.dfci.harvard.edu/"><strong>CARE</strong></a> &nbsp;
CARE is a software developed to enable large-scale inference of response biomarkers and drug combinations for targeted therapies using compound screen data.</li>
  <li><a href="http://tide.dfci.harvard.edu/"><strong>TIDE</strong></a> &nbsp;
TIDE is a gene expression biomarker to predict the clinical response to immune checkpoint blockade. TIDE also provides gene-wise associations with T-cell dysfunction and immunotherapy resistance, computed from a large amount of cancer genomics datasets.</li>
  <li><a href="https://github.com/liulab-dfci/2LNet"><strong>DrugCellNet</strong></a> &nbsp;
DrugCellNet is a dual-layer integrated cell line-drug network model, which uses both cell line similarity network (CSN) data and drug similarity network (DSN) data to predict the drug response of a given cell line using a weighted model.</li>
</ul>

<h3 id="crispr-analysis">CRISPR Analysis</h3>
<ul>
  <li><a href="http://bioconductor.org/packages/release/bioc/html/MAGeCKFlute.html"><strong>MAGeCKFlute</strong></a> &nbsp;
MAGeCKFlute is designed to support downstream analysis, utilizing the gene summary data provided through MAGeCK or MAGeCK-VISPR. Quality control, normalization, and screen hit identification for CRISPR screen data are performed in pipeline. Identified hits within the pipeline are categorized based on experimental design, and are subsequently interpreted by functional enrichment analysis.</li>
  <li><a href="http://bitbucket.org/liulab/mageck-vispr/"><strong>MAGeCK-VISPR</strong></a> &nbsp;
Model-based Analysis of Genome-wide CRISPR-Cas9 Knockout + Visualization of CRISPR Screens (MAGeCK-VISPR) is a comprehensive pipeline for analyzing and visualizing results from genome-scale CRISPR-Cas9 knockout screens.</li>
  <li><a href="https://bitbucket.org/liulab/mageck/src/master/"><strong>MAGeCK</strong></a> &nbsp;
Model-based Analysis of Genome-wide CRISPR-Cas9 Knockout (MAGeCK) is a computational tool to identify important genes from genome-scale CRISPR-Cas9 knockout screens.</li>
  <li><a href="http://nest.dfci.harvard.edu/"><strong>NEST</strong></a> &nbsp;
Network Essentiality Scoring Tool (NEST) is designed to predict gene essentiality based on protein interaction network and gene expression. It can also be used to enhance the quality of CRISPR screen results.  Publication submitted.</li>
  <li><a href="http://cistrome.org/crispr/"><strong>CRISPR-DO</strong></a>  &nbsp;
Genome-wide CRISPR gRNA design and optimization</li>
  <li><a href="http://cistrome.org/crispr-focus/"><strong>CRISPR-FOCUS</strong></a>  &nbsp;
A tool for designing customized sgRNA libray against user input gene list.</li>
  <li><a href="https://sourceforge.net/projects/spacerscoringcrispr/"><strong>SSC</strong></a> &nbsp;
Sequence Scan for CRISPR gRNA efficiency.</li>
</ul>

<h3 id="chip-seq-and-epigenetics-analysis">ChIP-Seq and Epigenetics Analysis</h3>
<ul>
  <li><b>LISA</b> <a href="http://lisa.cistrome.org/"><strong>web</strong></a> &nbsp; <a href="https://github.com/liulab-dfci/lisa2"><strong>command line</strong></a> &nbsp; 
leverages public ChIP-seq and chromatin accessibiity profiles in CistromeDB to discover the key transcription factors and chromatin regulators from user-provided differential expression gene list.</li>
  <li><a href="http://go.cistrome.org/"><strong>Cistrome-GO</strong></a> &nbsp;
Cistrome-GO is a webserver for functional enrichment analysis of transcription factor ChIP-seq peaks.</li>
  <li><a href="http://dbtoolkit.cistrome.org/"><strong>Cistrome DB Toolkit</strong></a> &nbsp;
Backed by the huge collection of Cistrome DB, the Toolkit allows users to quickly examine their hypothesis regarding gene regulation through public ChIP-seq (protein factors and histone marks), chromatin accessibility (DNase-seq and ATAC-seq) data. Three functions have been implemented in Cistrome DB Toolkit website.</li>
  <li><a href="http://rabit.dfci.harvard.edu/"><strong>RABIT</strong></a> &nbsp;
Regression Analysis with Background Integration (RABIT) is a very efficient feature selection algorithm. We applied RABIT to find tumor associated regulators in diverse cancer types.</li>
  <li><a href="http://cistrome.org/ap/"><strong>Cistrome</strong></a> &nbsp;
A web-based integrative platform for transcriptional regulation studies, from peak calling, correlation and association studies, expression analyses, to motif discovery.</li>
  <li><a href="http://cistrome.org/db"><strong>Cistrome DB</strong></a> &nbsp;
Collects and processes public ChIP-seq / DNase-seq data for quality control, visualization, and download.</li>
  <li><a href="https://github.com/taoliu/MACS"><strong>Model-based Analysis of ChIP-Seq (MACS)</strong></a> &nbsp;
A model-based algorithm for finding enriched regions in ChIP-Seq experiments.</li>
  <li><a href="http://cistrome.org/chilin/"><strong>ChiLin</strong></a> &nbsp;
ChiLin is an extensible software suite that is ideal for processing and quality control of large batches of ChIP-seq and DNase-seq data. Publication submitted.</li>
  <li><a href="http://cistrome.org/BETA"><strong>BETA</strong></a> &nbsp;
Targets analysis pipeline by integration of transcriptome and ChIP-seq data.</li>
  <li><a href="https://liulab-dfci.github.io/resources/software/BINOCh.zip"><strong>BINOCh</strong></a> &nbsp;
Binding Inference from Nucleosome Occupancy Changes.</li>
  <li><a href="https://liulab-dfci.github.io/resources/software/NPS.zip"><strong>Nucleosome Positioning from Sequencing (NPS)</strong></a> &nbsp;
A signal processing-based algorithm for identifying positioned nucleosomes fromsequencing experiments at the nucleosome level.</li>
  <li><a href="https://liulab-dfci.github.io/resources/software/CEAS.zip"><strong>Cis-regulatory Element Annotation System(CEAS) - Standalone version</strong></a> &nbsp;
A standalone Python package for analyzing ChIP-chip/seq data.</li>
  <li><a href="http://ceas.cbi.pku.edu.cn/"><strong>Cis-regulatory Element Annotation System(CEAS)</strong></a> &nbsp;
An integrated webserver for analyzing ChIP-chip data.</li>
</ul>

<h3 id="transcriptome-analysis">Transcriptome Analysis</h3>
<ul>
  <li><a href="https://bitbucket.org/feeldead/gfold"><strong>GFold</strong></a> &nbsp;
Generalized fold change for ranking differentially expressed genes from RNA-seq data.</li>
  <li><a href="https://liulab-dfci.github.io/resources/software/sageExpress.zip"><strong>SAGE-seq gene EXPRESSion analysis pipelines (sageExpress)</strong></a> &nbsp;
Gene expression analysis pipelines for SAGE-Seq including tag mapping, novel normalization method using empirical Bayes and differential gene analysis.</li>
</ul>

<h3 id="chip-chip-analysis">ChIP-chip Analysis</h3>
<ul>
  <li><a href="https://liulab-dfci.github.io/resources/software/MAT-3.07312009.tar.gz"><strong>Model-based Analysis of Tiling-array (MAT)</strong></a> &nbsp;
A model-based algorithm for finding enriched regions in ChIP-Chip experiments.</li>
  <li><a href="https://liulab-dfci.github.io/resources/software/MA2C.zip"><strong>Model-based Analysis of 2-Color Arrays (MA2C)</strong></a> &nbsp;
A model-based algorithm for analyzing 2-color microarrays.</li>
  <li><a href="https://liulab-dfci.github.io/resources/software/xMAN.tar.gz"><strong>extreme MApping of OligoNucleotides (xMAN)</strong></a> &nbsp;
A simple and efficient method for the mapping of millions of queryoligonucleotide fragments to the subject genome of any given length, at least an order of magnitude faster than other popular existing tools.</li>
  <li><a href="https://liulab-dfci.github.io/resources/software/HMMTiling.zip"><strong>Hidden Markov model for analyzing ChIP-chip experiments on genome tiling arrays (HMMTiling)</strong></a> &nbsp;
HMMTiling is a comprehensive software package for tiling array data analysis. It includes command line python applications for filtering, mapping, quantile-normalizing and enriched-region identification from ChIP-chip experiments on tiling arrays.</li>
  <li><a href="https://liulab-dfci.github.io/resources/software/MBR.zip"><strong>Microarray Blob Remover (MBR)</strong></a> &nbsp;
A microarray JAVA tool which allows rapid visualization, detection, and removalof blob-like defects as an initial quality control step.</li>
</ul>

<h3 id="transcription-factor-motif-finding">Transcription Factor Motif Finding</h3>
<ul>
  <li><a href="http://compareprospector.stanford.edu/"><strong>CompareProspector</strong></a> &nbsp;
A sequence motif-finding algorithm.</li>
  <li><a href="http://www.math.umass.edu/~conlon/mr.html"><strong>Motif Regressor</strong></a> &nbsp;
A program for finding DNA sequence motifs.</li>
  <li><a href="http://robotics.stanford.edu/~xsliu/MDscan/"><strong>MDscan</strong></a> &nbsp;
A fast and accurate motif finding algorithm with applications to chromatin immunoprecipitation microarray experiments.</li>
  <li><a href="http://robotics.stanford.edu/~xsliu/BioProspector/"><strong>BioProspector</strong></a> &nbsp;
Discovering conserved DNA motifs in upstream regulatory regions of co-expressedGenes.</li>
</ul>

  </div>
</div>



    <footer>
  <div class="container beautiful-jekyll-footer">
    <div class="row">
      <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
        <ul class="list-inline text-center footer-links"><li><a href="https://liulab-dfci.github.io/feed.xml" title="RSS"><span class="fa-stack fa-lg" aria-hidden="true">
                  <i class="fa fa-circle fa-stack-2x"></i>
                  <i class="fa fa-rss fa-stack-1x fa-inverse"></i>
                </span>
                <span class="sr-only">RSS</span>
              </a>
            </li><li><a href="https://github.com/liulab-dfci" title="GitHub"><span class="fa-stack fa-lg" aria-hidden="true">
                  <i class="fa fa-circle fa-stack-2x"></i>
                  <i class="fa fa-github fa-stack-1x fa-inverse"></i>
                </span>
                <span class="sr-only">GitHub</span>
              </a>
            </li><li><a href="https://twitter.com/xshirleyliu" title="Twitter"><span class="fa-stack fa-lg" aria-hidden="true">
                  <i class="fa fa-circle fa-stack-2x"></i>
                  <i class="fa fa-twitter fa-stack-1x fa-inverse"></i>
                </span>
                <span class="sr-only">Twitter</span>
              </a>
            </li></ul>
      <p class="copyright text-muted">
      Xiaole Shirley Liu Lab
      &nbsp;â€¢&nbsp;
      2023

      

      
      </p>
          <!-- Please don't remove this, keep my open source work credited :) -->
    <p class="theme-by text-muted">
      Theme by
      <a href="https://deanattali.com/beautiful-jekyll/">beautiful-jekyll</a>
    </p>
      </div>
    </div>
  </div>
</footer>

  
    


  
    <!-- doing something a bit funky here because I want to be careful not to include JQuery twice! -->
    
      <script>
      	if (typeof jQuery == 'undefined') {
          document.write('<script src="/js/jquery-1.11.2.min.js"></scr' + 'ipt>');
      	}
      </script><script src="./Software_files/jquery-1.11.2.min.js"></script>
    
  
    <!-- doing something a bit funky here because I want to be careful not to include JQuery twice! -->
    
      <script src="./Software_files/bootstrap.min.js"></script>
    
  
    <!-- doing something a bit funky here because I want to be careful not to include JQuery twice! -->
    
      <script src="./Software_files/main.js"></script>
    
  






  
  

</body></html>